---
title: "*Clostridium difficile* changes murine gut communities"
csl: isme.csl
output:
 word_document:
  keep_md: true
  reference_docx: manuscript_format.docx
bibliography: references.bib
---

**Authors:** Matthew L. Jenior, Jhansi L. Leslie, Vincent B. Young, and Patrick D. Schloss^\*^


```{r startup, echo=F, message=F, warning=F, cache=F}

# Load dependencies
deps <- c('shape', 'vegan', 'klaR', 'igraph', 'matrixStats', 'flux', 'knitr','plyr')
for (dep in deps){
 if (dep %in% installed.packages()[,"Package"] == FALSE){
  install.packages(as.character(dep), quiet=TRUE);
 }
 library(dep, verbose=FALSE, character.only=TRUE)
}

# Define function for formatting growth curves for statistical analysis
format_curve <- function(raw_exp_data, exp_group, raw_control_data){
 formatted_data <- c()
 control_data <- c()
 for (time in 1:nrow(raw_exp_data)){
  temp_exp <- cbind(exp_group, time, raw_exp_data[time,])
  formatted_data <- rbind(formatted_data, temp_exp)
  temp_control <- cbind('control', time, raw_control_data[time,])
  control_data <- rbind(control_data, temp_control)
 }
 formatted_data <- as.data.frame(rbind(control_data, formatted_data))
 colnames(formatted_data) <- c('substrate','time','od')
 formatted_data$od <- as.numeric(as.character(formatted_data$od))

 return(formatted_data)
}

# Mouse metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=T, row.names=1)
# Figure 1 data
cfu <- read.delim('data/wetlab_assays/cfu.dat', sep='\t', header=T)
toxin <- read.delim('data/wetlab_assays/toxin_titer.dat', sep='\t', header=T)



```


###Abstract



###Introduction



###Results


```{r abx_table, echo=F, message=F, warning=F, cache=F}

#kable()
```







***C. difficile* differentially expresses virulence factors across distinct gut microbiome.**

Utilizing 

Conventionally-reared SPF mice were treated with one of three different antibiotics. These included streptomycin, cefoperazone, and clindamycin (Table 1 and Fig. S1), which were selected because they each have distinct, significant impacts on the structure and diversity of the cecal microbiome (Fig. S2A and S2B). This toxigenic strain of C. difficile was chosen for its moderate clinical severity in mouse models (24) and well-annotated genome (25). After infection, we measured sporulation and toxin production at 18 hours post innoculation. That time point corresponded with when another laboratory strain of C. difficile reached its maximum vegetative cell density in the cecum with limited sporulation (26). 

There was not a significant difference in the number of vegetative cells between any susceptible condition tested (Fig. 1A). All antibiotic treated and ex-GF mice were colonized to ~1×108 colony forming units (cfu) per gram of cecal content, while untreated mice maintained colonization resistance to C. difficile (Table 1). Despite having the same number of vegetative C. difficile cells, more spores were detected in ex-GF mice than in the antibiotic pretreated mice (P = 0.003, 0.004, and 0.003; Fig. 1B). There was also a significantly higher toxin titer in ex-GF animals than any other colonized group (all P < 0.001), with slight variation between antibiotic pretreatment groups (Fig. 1C). These results showed that C. difficile colonized different communities to consistently high levels. In addition, colonization in the context of different microbiomes resulted in moderate differences in the production of C. difficile virulence factors.

at the 18 hour time point communities in Streptomycin-pretreated animals exhibit more significant change in structure (Fig 1C & Fig. S2A.




***C. difficile* colonization changes the structure the surrounding cecal bacterial community.**

Next we determined that *C. difficile* colonization, in the absence of sequences contributed by *C. difficile* itself, altered the structure of the surrounding community (Fig. S2). Apparent changes were present in all antibiotic pretreatment groups, but associated with in streptomycin and clindamycin were significant (*p* < 0.01**, Fig. S2c & S2d). In order to determine which OTUs were most responsible for these shifts we performed pair-wise, corrected Wilcoxon tests on the relative abundance of each OTU in either *C. difficile* or mock-infected cecal communities (Fig. 1d-1f). Each group contained distinct sets of OTUs, save for Lactobacillus (OTU1) which had significant changes in both cefoperazone and clindamycin pretreatments (*p* < 0.05*). Interestingly, the type of changes were not consistent across all OTUs in that some would decrease following *C. difficile* colonization while others instead increased in abundance. These findings are suggestive that *C. difficile* may be differentially interacting these respective taxa either synergistically or antagonistically. Furthermore, the fact the which OTUs are affected differs between treatments also suggests the *C. difficile* may alter behavior between environments to best augment its own colonization.




***C. difficile* colonization alters community-level expression distinctly between perturbed environments**




In order to more closely examine the response of ... we performed metatranscriptomic sequencing









**Modeling competition for nutrients across the microbiome**

To more clearly integrate community-level transcriptomic data, we chose to leverage methods for genome-scale metabolic modeling previouslt developed in our laboratory [@Jenior2017]. To generate annotated genomes for discrete bacterial taxa on which to map transcriptomic reads, we implemented clustering of our assembled contigs based on abundance and composition [CONCOCT reference]. Strongly clustered groups of sequences were treated as individual species and a separate metabolic model with metabolite importance scores were calculated for each (Fig. S____ CONCOCT clusters in supplement for this section). Within each community, we then compared pair-wise ratios of metabolite importance scores for each possible interspecies interaction utilizing evenly rarified transcriptomic abundances between the models (Fig. 4A). This provided a metric that reflected the level of competition for a given metabolite existed between particular pairs of bacterial taxa. We then went on to perform cumulative calculation of importance for each metabolite across all species in a given community (Fig. 4B). Values from this analysis give and estimation of the most highly sought after metabolites in the communities tested. Combined, this approach allowed for predicition of nutrient niche segregation between species in the cecal bacterial community and how *C. difficile* altered this landscape to establish its own metabolic strategy.


Add panels to fig 4 about individual species importances and high demand metabolites.
 - paragraph discussing these results here


```{r figure_5, echo=F, warning=F, message=F, results='hide', cache=F}

# Metabolomic data
metabolome <- read.delim('data/wetlab_assays/metabolomics.tsv', sep='\t', header=T)
annotation <- metabolome[,1:5]
metabolome$SUPER_PATHWAY <- NULL
metabolome$SUB_PATHWAY <- NULL
metabolome$PUBCHEM <- NULL
metabolome <- subset(metabolome, KEGG != 'NA')
metabolome <- metabolome[match(unique(metabolome$KEGG), metabolome$KEGG),]
rownames(metabolome) <- metabolome$KEGG
metabolome$KEGG <- NULL
metabolome$BIOCHEMICAL <- NULL
annotation <- subset(annotation, KEGG != 'NA')
annotation <- annotation[match(unique(annotation$KEGG), annotation$KEGG),]
rownames(annotation) <- annotation$KEGG
annotation$KEGG <- NULL
annotation$SUB_PATHWAY <- NULL
annotation$PUBCHEM <- NULL
annotation$BIOCHEMICAL <- NULL
annotation$SUPER_PATHWAY <- gsub('_',' ', annotation$SUPER_PATHWAY)

# Combine with metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=T, row.names=1)
metabolome <- merge(t(metabolome), metadata, by='row.names')
rm(metadata)
rownames(metabolome) <- metabolome$Row.names
metabolome$Row.names <- NULL
metabolome <- subset(metabolome, abx != 'none')
metabolome_630 <- subset(metabolome, infection == '630')
metabolome_630$cage <- NULL
metabolome_630$mouse <- NULL
metabolome_630$gender <- NULL
metabolome_630$type <- NULL
metabolome_630$infection <- NULL
metabolome_630 <- aggregate(metabolome_630[, 1:399], list(metabolome_630$abx), median)
rownames(metabolome_630) <- metabolome_630$Group.1
metabolome_630$Group.1 <- NULL
metabolome_630 <- as.data.frame(t(metabolome_630))
colnames(metabolome_630) <- c('cef_630','clinda_630','gf_630','strep_630')
metabolome_mock <- subset(metabolome, infection == 'mock')
metabolome_mock$cage <- NULL
metabolome_mock$mouse <- NULL
metabolome_mock$gender <- NULL
metabolome_mock$type <- NULL
metabolome_mock$infection <- NULL
metabolome_mock <- aggregate(metabolome_mock[, 1:399], list(metabolome_mock$abx), median)
rownames(metabolome_mock) <- metabolome_mock$Group.1
metabolome_mock$Group.1 <- NULL
metabolome_mock <- as.data.frame(t(metabolome_mock))
colnames(metabolome_mock) <- c('cef_mock','clinda_mock','gf_mock','strep_mock')

# Calculate ratio of mock to infected concentrations
metabolome_mock <- 10 ^ metabolome_mock
metabolome_630 <- 10 ^ metabolome_630
metabolome <- metabolome_mock / metabolome_630
metabolome <- log10(metabolome)
metabolome <- merge(metabolome, annotation, by='row.names')
rownames(metabolome) <- metabolome$Row.names
metabolome$Row.names <- NULL
colnames(metabolome) <- c('cefoperazone_conc', 'clindamycin_conc', 'streptomycin_conc', 'pathway')
rm(annotation, metabolome_mock, metabolome_630)

# Merge importances to one table
cef_importances <- read.delim('data/metabolic_models/cefoperazone_630.bipartite.files/importances.tsv', sep='\t', header=T, row.names=1)
cef_importances$p_value <- NULL
clinda_importances <- read.delim('data/metabolic_models/clindamycin_630.bipartite.files/importances.tsv', sep='\t', header=T, row.names=1)
clinda_importances$p_value <- NULL
clinda_importances$Compound_name <- NULL
strep_importances <- read.delim('data/metabolic_models/streptomycin_630.bipartite.files/importances.tsv', sep='\t', header=T, row.names=1)
strep_importances$p_value <- NULL
strep_importances$Compound_name <- NULL
importances <- merge(cef_importances, clinda_importances, by='row.names')
rownames(importances) <- importances$Row.names
importances$Row.names <- NULL
importances <- merge(importances, strep_importances, by='row.names')
rownames(importances) <- importances$Row.names
importances$Row.names <- NULL
colnames(importances) <- c('Compound_name', 'cefoperazone_score', 'clindamycin_score', 'streptomycin_score')
importances$Compound_name <- gsub('_',' ', importances$Compound_name)
rm(cef_importances, clinda_importances, strep_importances, gf_importances)

# Merge metabolome medians and importance values
combined <- merge(importances, metabolome, by='row.names')
rownames(combined) <- combined$Row.names
combined$Row.names <- NULL
rm(importances, metabolome)

# Separate treatment groups
cef <- as.data.frame(cbind(combined$cefoperazone_score, combined$cefoperazone_conc, combined$pathway))
rownames(cef) <- rownames(combined)
colnames(cef) <- c('score', 'conc', 'pathway')
cef$name <- combined$Compound_name
cef$score <- as.numeric(as.vector(cef$score))
cef$conc <- as.numeric(as.vector(cef$conc))
cef <- subset(cef, cef[,1] != 0) # remove metabolites with 0 importance
clinda <- as.data.frame(cbind(combined$clindamycin_score, combined$clindamycin_conc, combined$pathway))
rownames(clinda) <- rownames(combined)
colnames(clinda) <- c('score', 'conc', 'pathway')
clinda$name <- combined$Compound_name
clinda$score <- as.numeric(as.vector(clinda$score))
clinda$conc <- as.numeric(as.vector(clinda$conc))
clinda <- subset(clinda, clinda[,1] != 0) # remove metabolites with 0 importance
strep <- as.data.frame(cbind(combined$streptomycin_score, combined$streptomycin_conc, combined$pathway))
rownames(strep) <- rownames(combined)
colnames(strep) <- c('score', 'conc', 'pathway')
strep$name <- combined$Compound_name
strep$score <- as.numeric(as.vector(strep$score))
strep$conc <- as.numeric(as.vector(strep$conc))
strep <- subset(strep, strep[,1] != 0) # remove metabolites with 0 importance
combined <- rbind(strep, cef, clinda)

# Fit to general linear models and identify outliers (L1 regression)
strep_fit <- glm(conc ~ score, data=strep)
strep$residuals <- residuals(strep_fit)
strep$residuals <- (strep$residuals / sd(strep$residuals))^2
strep_outliers <- strep[strep$residuals > 1.5, ]
cef_fit <- glm(conc ~ score, data=cef)
cef$residuals <- residuals(cef_fit)
cef$residuals <- (cef$residuals / sd(cef$residuals))^2
cef_outliers <- cef[cef$residuals > 1.5, ]
clinda_fit <- glm(conc ~ score, data=clinda)
clinda$residuals <- residuals(clinda_fit)
clinda$residuals <- (clinda$residuals / sd(clinda$residuals))^2
clinda_outliers <- clinda[clinda$residuals > 1.5, ]
combined_fit <- glm(conc ~ score, data=combined)
combined$residuals <- residuals(combined_fit)
combined$residuals <- (combined$residuals / sd(combined$residuals))^2
combined_outliers <- combined[combined$residuals > 1.5, ]

# Calculate stats
corr_m <- round(c(strep_fit$coefficients[[2]],cef_fit$coefficients[[2]],clinda_fit$coefficients[[2]],combined_fit$coefficients[[2]]), digits=3)
corr_m_strep <- corr_m[1]
corr_m_cef <- corr_m[2]
corr_m_clinda <- corr_m[3]
corr_m_combined <- corr_m[5]

corr_r <- round(c(cor.test(strep[,1], strep[,2], method='spearman', exact=FALSE)$estimate,
        cor.test(cef[,1], cef[,2], method='spearman', exact=FALSE)$estimate,
        cor.test(clinda[,1], clinda[,2], method='spearman', exact=FALSE)$estimate,
        cor.test(combined[,1], combined[,2], method='spearman', exact=FALSE)$estimate), digits=3)
corr_r_strep <- corr_r[1]
corr_r_cef <- corr_r[2]
corr_r_clinda <- corr_r[3]
corr_r_combined <- corr_r[5]

corr_p <- round(c(cor.test(strep[,1], strep[,2], method='spearman', exact=FALSE)$p.value,
        cor.test(cef[,1], cef[,2], method='spearman', exact=FALSE)$p.value,
        cor.test(clinda[,1], clinda[,2], method='spearman', exact=FALSE)$p.value,
        cor.test(combined[,1], combined[,2], method='spearman', exact=FALSE)$p.value), digits=3)
corr_p_strep <- corr_p[1]
corr_p_cef <- corr_p[2]
corr_p_clinda <- corr_p[3]
corr_p_combined <- corr_p[5]

```


**Multi-omic integration supports predictive value of network platform for predict changes in metabolite concentrations**

More effective way to integrate metatranscriptome with metabolome - unified global correlation of gene expression and change in metabolite concentrations neglects the activity of individual species


Old text from previous paper:
Moving beyond specific metabolite analysis, we wanted to determine how well metabolite importance scores related to the relative concentration of all metabolites in a given environment. To most effectively combine metabolite importance scores with the affiliated concentration data, we decided to calculate the median change in relative concentration for each metabolite measured between corresponding mock and *C. difficile*-infected conditions. This was accomplished by dividing the median log10-scaled intensity of each metabolite in mock-infected mice by its corresponding intensity in *C. difficile*-infected mice to give the delta median scaled intensity for each metabolite. In this way, larger values indicated those metabolites that decreased in concentration when *C. difficile* colonized. This also resulted in a single value we could combine with importance scores for each metabolite and perform a linear correlation analysis. Positive correlations indicated that the metabolites predicted to be consumed by *C. difficile* through transcript-informed metabolic networks have a concomitant decrease in the metabolome. This relationship would also imply that those metabolites with negative importance scores and increased concentration in infected animals are being produced by *C. difficile*. 

We applied this approach first to a pooled analysis of all groups to maximize our ability to validate the predictive capacity of the modeling algorithm (Fig. 6A). This revealed a significant positive correlation (*P* = `r corr_p_combined`) and supported our method for identifying metabolites that are most likely to be consumed by *C. difficile* based on transcriptional activity. This technique also allowed us to analyze outliers from the generalized linear model to assess if they are explained by other known patterns of *C. difficile* biology. Applying least-squared residuals analysis, we found that 16 metabolites were determined to be outliers and were all classified 1 of 3 separate KEGG annotation categories; amino acids, carbohydrates, and nucleotides (Table S5). We went on to identify the majority subset of 12 outlier metabolites that were important and significantly decreased in their respective condition were annotated as growth substrates or growth enhancers of *C. difficile* [@Karasawa1995]. This included L-proline, L-threonine, N-acetyl-D-glucosamine, D-fructose, and galactitol. These results were interpretted as not a failure of our importance algortihm, but as a possible underestimation of enzyme efficiency for the consumption these metabolites which further supports our approach as a way to determine real *in vivo* metabolic strategies for *C. difficile*. We also explored those metabolites that were more likely to be produced based on importance scores but decreased during infection. These were the nucleotides adenine, deoxyadenosine, and xanthine. *C. difficile* has been shown to possess the capacity for their synthesis [@Karlsson2008], but also may have the ability to catabolize these purines to participate downstream in Stickland reactions [@Hartwich2012]. The findings here may indicate that *C. difficile* utilizes both environmental and synthesized pools of the molecules at a faster rate than they are produced by the bacterium alone and may reinforce the centrality of amino acid fermentation to *C. difficile* *in vivo* metabolism.

We then directed the analysis toward each individual infection condition tested. In streptomycin-pretreatment (Fig. 6B) and cefoperazone-pretreatment (Fig. 6C) the correlations did not achieve significance (*P* = `r corr_p_strep` & `r corr_p_cef`), but remained positive. Analysis of outliers revealed that in streptomycin-pretreatment both the host-derived aminoglycan N-acetyl-D-glucosamine and L-proline were implicated as most likely to be utilized by *C. difficile* while L-proline was the only growth substrate outlier in this way in cefoperazone-pretreated mice. Both clindamycin-pretreatment (Fig. 6D) and ex-GF mice (Fig. 6E) instead demonstrated a significant correlation between importance scores and delta median scaled intensity (*P* = `r corr_p_clinda` & `r corr_p_gf`). Although clindamycin-pretreatment was associated with most outlier metabolites, both groups also shared a similar trend of carbohydrates and Stickland fermentation substrates being important and decreased in infected conditions. Proline specifically was found to be both highly important and decreased in concentration across all infections. In combination with the previously mentioned patterns, this further supports Stickland fermentation as a core metabolic strategy of *C. difficile* in the gut which agrees with previous *in vitro* findings [@Jackson2006]. Together, these data supported network-derived importance scores as well as our hypothesis that *C. difficile* adapted portions of its carbon source metabolism in distinct susceptible environments in addition to certain highly consistent strategies.

###Discussion



###Materials and Methods

**Animal care and antibiotic administration** Adapted from the previously described model [@Theriot2011], six-to-eight week-old SPF C57BL/6 mice were obtained from a single breeding colony maintained at the University of Michigan for all experiments. Six-to-eight week-old GF C57BL/6 mice were obtained from a single breeding colony maintained at the University of Michigan and fed Laboratory Rodent Diet 5001 from LabDiet for all experiments. All animal protocols were approved by the University Committee on Use and Care of Animals at the University of Michigan and carried out in accordance with the approved guidelines. Specified SPF animals were administered one of three antibiotics; cefoperazone, streptomycin, or clindamycin (Table 1). Cefoperazone (0.5 mg/ml) and streptomycin (5.0 mg/ml) were administered in distilled drinking water *ad libitum* for 5 days with 2 days recovery with untreated distilled drinking water prior to infection. Clindamycin (10 mg/kg) was given via intraperitoneal injection 24 hours before time of infection.

***C. difficile* infection and necropsy.** All *C. difficile* strain 630 spores were prepared from a single large batch whose concentration was determined a week prior to challenge. On the day of challenge, 1×10^3^ *C. difficile* spores were administered to mice via oral gavage in phosphate-buffered saline (PBS) vehicle. Subsequent quantitative plating for cfu was performed to ensure correct dosage. Mock-infected animals were given an oral gavage of 100 μl PBS at the same time as those mice administered *C. difficile* spores. 18 hours following infection, mice were euthanized by carbon dioxide asphyxiation and necropsied to obtain the cecal contents. Two 100 μl aliquots were immediately flash frozen for later DNA extraction and toxin titer analysis respectively. A third 100 μl aliquot was quickly transferred to an anaerobic chamber for quantification of *C. difficile* abundance. The remaining content in the ceca (approximately 1 mL) was mixed with 1 mL of sterile PBS in a stainless steel mortar housed in a dry ice and ethanol bath. The cecal content of 9 mice from 3 cages was pooled into the mortar. Pooling cecal contents was necessary so that there would be a sufficient quantity of high quality rRNA-free RNA for deep sequencing. The pooled content was then finely ground and stored at -80° C for subsequent RNA extraction.

***C. difficile* cultivation and quantification** Cecal samples were weighed and serially diluted under anaerobic conditions (6% H, 20% CO~2~, 74% N~2~) with anaerobic PBS. Differential plating was performed to quantify both *C. difficile* spores and vegetative cells by plating diluted samples on CCFAE plates (fructose agar plus cycloserine, cefoxitin, and erythromycin) at 37° C for 24 hours under anaerobic conditions [@Wilson1982]. It is important to note that the germination agent taurocholate was omitted from these plates to quantify only vegetative cells. In parallel, undiluted samples were heated at 60° C for 30 minutes to eliminate vegetative cells and leave only spores [@Sorg2010]. These samples were serially diluted under anaerobic conditions in anaerobic PBS and plated on CCFAE with taurocholate at 37° C for 24 hours. Plating was simultaneously done for heated samples on CCFAE to ensure all vegetative cells had been eliminated.

***C. difficile* toxin titer assay** To quantify the titer of toxin in the cecum a Vero cell rounding assay was performed as in [@Leslie2015]. Briefly, filtered-sterilized cecal content was serially diluted in PBS and added to Vero cells in a 96-well plate. Plates were viewed after 24 hour incubation for cell rounding. A more detailed protocol with product information can be found at: https://github.com/SchlossLab/Jenior_Modeling_mBio_2017/blob/master/protocols/toxin_assay/Verocell_ToxinActivity_Assay.Rmd

**16S rRNA gene sequencing and curation** DNA was extracted from approximately 50 mg of cecal content from each mouse using the PowerSoil-htp 96 Well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf). The V4 region of the bacterial 16S rRNA gene was amplified using custom barcoded primers and sequenced as described previously using an Illumina MiSeq sequencer [@Kozich2013]. All 63 samples were sequenced on a single sequencing run. The 16S rRNA gene sequences were curated using the mothur software package (v1.36), as described previously [@Kozich2013]. In short, paired-end reads were merged into contigs, screened for quality, aligned to SILVA 16S rRNA sequence database, and screened for chimeras. Sequences were classified using a naive Bayesian classifier trained against a 16S rRNA gene training set provided by the Ribosomal Database Project (RDP) [@Wang2007]. Curated sequences were clustered into operational taxonomic units (OTUs) using a 97% similarity cutoff with the average neighbor clustering algorithm. The number of sequences in each sample was rarefied to 2,500 per sample to minimize the effects of uneven sampling.

**DNA extraction, shotgun library preparation, and sequencing** 


**Metagenomic assembly and annotation.**


**RNA extraction, shotgun library preparation, and sequencing** To generate enough mRNA biomass contributed by *C. difficile*, we pooled cecal content from all mouse replicates into a single large isolation for each treatment group. Pooling was performed in a sterile stainless steel mortar resting in dry ice and a small amount of 100% ethanol. After all content for the given group was added, the sample was ground with a sterile pestle to a fine powder and scraped into a sterile 50 ml polypropylene conical tube. Samples were stored at -80° C until the time of extraction. Immediately before RNA extraction, 3 ml of lysis buffer (2% SDS, 16 mM EDTA and 200 mM NaCl) contained in a 50 ml polypropylene conical tube was first heated for 5 minutes in a boiling water bath [@Lopez-Medina2011]. The hot lysis buffer was added to the frozen and ground cecal content. The mixture was boiled with periodic vortexing for another 5 minutes. After boiling, an equal volume of 37° C acid phenol/chloroform was added to the cecal content lysate and incubated at 37° C for 10 minutes with periodic vortexing. The mixture was the centrifuged at 2,500 x g at 4° C for 15 minutes. The aqueous phase was then transferred to a sterile tube and an equal volume of acid phenol/chloroform was added. This mixture was vortexed and centrifuged at 2,500 x g at 4° for 5 minutes. The process was repeated until aqueous phase was clear. The last extraction was performed with chloroform/isoamyl alcohol to remove acid phenol. An equal volume of isopropanol was added and the extracted nucleic acid was incubated overnight at -20° C. The following day the sample was centrifuged at 12000 x g at 4° C for 45 minutes. The pellet was washed with 0° C 100% ethanol and resuspended in 200 μl of RNase-free water. Following the manufacturer's protocol, samples were then treated with 2 μl of Turbo DNase for 30 minutes at 37° C. RNA samples were retrieved using the Zymo Quick-RNA MiniPrep according the manufacturer's protocol. Completion of the reaction was assessed using PCR for the V4 region of the 16S rRNA gene for 30 cycles (Kozich, 2013). Quality and integrity of RNA was measured using the Agilent RNA 6000 Nano kit for total prokaryotic RNA. The Ribo-Zero Gold rRNA Removal Kit Epidemiology was then used to deplete prokaryotic and eukaryotic rRNA from the samples according the manufacturer's protocol. Prior to library construction, quality and integrity as measured again using the Agilent RNA 6000 Pico Kit. Stranded RNA-Seq libraries were made constructed with the TruSeq Total RNA Library Preparation Kit v2, both using the manufacturer's protocol. The Agilent DNA High Sensitivity Kit was used to measure concentration and fragment size distribution before sequencing. High-throughput sequencing was performed by the University of Michigan Sequencing Core in Ann Arbor, MI. For all groups, sequencing was repeated across 4 lanes of an Illumina HiSeq 2500 using the 2x50 bp chemistry.

**Sequence curation, read mapping, and normalization.** Raw transcript sequencing read curation was performed in a two step process. Residual 5’ and 3’ Illumina adapter sequences were trimmed using CutAdapt [@Martin2013] on a per library basis. Reads were quality trimmed using Sickle (Joshi, 2011) on the default settings. An average of \~300,000,000 total reads (both paired and orphaned) remained after quality trimming. Mapping was accomplished using Bowtie2 [@Langmead2009] and the default stringent settings. \~1,600,000 reads in sample each mapped to the annotated nucleotide gene sequences of *PeptoClostridium difficile* str. 630 from the KEGG: Kyoto Encyclopedia of Genes and Genomes [@Ogata1999]. Optical and PCR duplicates were then removed using Picard MarkDuplicates (http://broadinstitute.github.io/picard/), leaving ~150,000 reads per sample for final analysis. The remaining mappings were converted to idxstats format using Samtools [@Li2009] and the read counts per gene were tabulated. Discordant pair mappings were discarded and counts were then normalized to read length and gene length to give a per base report of gene coverage. Unless indicated otherwise, each collection of reads was then 1000-fold iteratively subsampled to 90% of the lowest sequence total within each analysis, and a median expression value for each gene was calculated.

**Metabolic modeling.**

Algorithm



**Quantification of *in vivo* metabolite relative concentrations.** Metabolomic analysis performed by Metabolon (Durham, NC), a brief description of their methods is as follows. All methods utilized a Waters ACQUITY ultra-performance liquid chromatography (UPLC) and a Thermo Scientific Q-Exactive high resolution/accurate mass spectrometer interfaced with a heated electrospray ionization (HESI-II) source and Orbitrap mass analyzer at 35,000 mass resolution. Samples were dried then reconstituted in solvents compatible to each of the four methods. The first, in acidic positive conditions using a C18 column (Waters UPLC BEH C18-2.1x100 mm, 1.7 µm) using water and methanol, containing 0.05% perfluoropentanoic acid (PFPA) and 0.1% formic acid (FA). The second method was identical to the first but was chromatographically optimized for more hydrophobic compounds. The third approach utilized a basic negative ion optimized conditions using a separate dedicated C18 column. Basic extracts were gradient eluted from the column using methanol and water, however with 6.5mM Ammonium Bicarbonate at pH 8. Samples were then analyzed via negative ionization following elution from a hydrophilic interaction chromatography column (Waters UPLC BEH Amide 2.1x150 mm, 1.7 µm) using a gradient consisting of water and acetonitrile with 10mM Ammonium Formate, pH 10.8. The MS analysis alternated between MS and data-dependent MS n scans using dynamic exclusion. The scan range varied slighted between methods but covered 70-1000 m/z. Library matches for each compound were checked for each sample and corrected if necessary. Peaks were quantified using area under the curve.

**Statistical methods.** All statistical analyses were performed using R (v.3.2.0). Significant differences between community structure of treatment groups from 16S rRNA gene sequencing were determined with AMOVA in the mothur software package. Significant differences of Inv. Simpson diversity, cfu, toxin titer, and metabolite concentrations were determined by Wilcoxon signed-rank test with Benjamini-Hochberg correction. Undetectable points used half the limit of detection for all statistical calculations. Significant differences for growth curves compared to no carbohydrate control (+ amino acids) were calculated using 1-way ANOVA with Benjamini-Hochberg correction.


###Funding Information

This work was supported by funding from the National Institutes of Health to PDS (R01GM099514, P30DK034933, U19AI09087, and U01AI124255), VBY (P30DK034933, U19AI09087, and U01AI124255), a Translational Research Education Certificate grant to JLL (MICHR; UL1TR000433), and was partially supported by a predoctoral fellowship from the Cellular Biotechnology Training Program to MLJ (T32GM008353).


###Acknowledgements

The authors would like to acknowledge Charles Koumpouras for assistance with DNA extractions and metabolomic sample preparation. We would also like to acknowledge members of the University of Michigan Germfree Mouse Center, University of Michigan Sequencing Core, and Metabolon for their assistance in experimental design, execution, and data collection. Pooled and quality trimmed transcriptomic read data and experiment metadata are available through the NCBI Sequence Read Archive (SRA; PRJNA354635 and ). Data processing steps for beginning from raw sequence data to the final manuscript are hosted at 

**Author Affiliations**
**Department of Microbiology and Immunology, University of Michigan, Ann Arbor, Michigan.**
Matthew L. Jenior, Jhansi L. Leslie, & Patrick D. Schloss Ph.D.

**Department of Internal Medicine & Infectious Diseases, University of Michigan Medical Center, Ann Arbor, Michigan.**
Vincent B. Young M.D. Ph.D.

**Author Contributions**
M.L.J. conceived, designed and performed experiments, analyzed data, and drafted the manuscript. J.L.L. performed experiments and analyzed data. V.B.Y. contributed to the manuscript. P.D.S. interpreted data and contributed the manuscript.
The authors declare no conflicts of interest.

**Corresponding author**
Correspondence to [Patrick D. Schloss](pdschloss@umich.edu)


###Figure Legends

Abx table

**Figure 1 | ** Description of animal models and changes to community from abx



**Figure 2 | ** Correlation of 16S and metabolome (species level) A) 16s nmds B) metabolome nmds C) combined analysis

**Figure 3 | *C. difficile* colonization alters community-level gene expression differently between antibiotic pretreatements.** All points represent unique genes from metagenomic assembly and each metatranscriptome is normalized to their respective metganomic coverage. Point position is determined by the log~2~-transformed expression level of each gene between *C. difficile*-infected and mock-infected conditions. Points are colored based on the annotated genus of origin. **(a)** Streptomycin-pretreatment, **(b)** Cefoperazone-pretreatment, and **(c)** Clindamycon-pretreatment. Outliers were defined by least-squared residuals analysis. **(d)** Top 5 expression of KEGG pathways in each condition.

**Figure 4 | ** Community-level modeling schematic

**Figure 5 | ** 
- Most important metabolites at community level
- most important to each community member
- global correlation of importances and metabolomics



**Supplementary Figure 1 | Rarefaction curves for reads?**

**Supplementary Figure 2 | NMDS including untreated controls and within group ordinations**

**Supplementary Figure 3 | **

**Supplementary Figure 4 | CONCOCT clustering**


### References
