---
title: "*Clostridium difficile* alters the structure and metabolism of distinct cecal microbiomes during acute infection to promote sustained colonization"
csl: msphere.csl
output:
 word_document:
  keep_md: true
  reference_docx: manuscript_format.docx
bibliography: references.bib
---

**Authors:** Matthew L. Jenior, Jhansi L. Leslie, Vincent B. Young, and Patrick D. Schloss^\*^


**Abstract**

Susceptibility to *Clostridium difficile* infection is primarily associated with previous exposure to antibiotics, which compromise the structure and function of the gut bacterial community. Specific antibiotic classes correlate more strongly with recurrent or persistent *C. difficile* infection. As such, we utilized a mouse model of infection to explore the effect of distinct antibiotic classes on the impact that infection has on community-level transcription and metabolic signatures shortly following pathogen colonization and how those changes may associate with persistence of *C. difficile*. Untargeted metabolomic analysis revealed that *C. difficile* infection has significantly larger impacts on the metabolic environment across cefoperazone and streptomycin-pretreated mice, which become persistently colonized compared to clindamycin-pretreated mice where infection quickly becomes undetectable. Through metagenome-enabled metatranscriptomics we observed that the infected microbial communities were greatly reduced in transcript for genes associated with carbon and energy acquisition, suggesting niches occupied instead by *C. difficile*. Furthermore, the largest degree of change in transcription were seen in the least abundant species within the respective groups indicating that *C. difficile* may "attack the loser" in gut environments where sustained infection occurs more readily. Overall, our results suggest that *C. difficile* is able to restructure the nutrient-niche landscape in the gut in order to promote persistent infection.

**Importance**

*Clostridium difficile* has become the most common single cause of hospital-acquired infection over the last decade in the United States and colonization resistance to the nosocomial pathogen is primarily driven by the gut microbiota. This community is also involved in clearing the infection as the community recovers from perturbation. As distinct antibiotics are associated with different risk levels for CDI, we utilized a mouse model of infection with 3 separate antibiotic pretreatment regimes to generate alternative gut microbiomes that each allowed for *C. difficile* colonization but vary in clearance rate. To assess community-level dynamics, we implemented an integrative multi-omic approach that revealed infection significantly shifted many aspects of the gut ecosystem. This revealed a trend that the degree to which this change occurred inversely correlated with clearance during the first six days of infection, suggesting that *C. difficile* may differentially modify the gut environment to promote persistence. This is the first time metagenome-enabled metatranscriptomics have been employed to study the behavior of a host-associated microbiota in response to an infection. Our results allow for previously unseen understanding of the ecology associated with *C. difficile* infection and provides groundwork for identification of context-specific probiotic therapies.


**Introduction**

One of the many beneficial functions provided by the indigenous gut bacterial community is its ability to protect the host from infection by pathogens [@Vollaard1994]. This attribute, termed colonization resistance, is one of the main mechanisms that protect healthy individuals from the gastrointestinal pathogen *Clostridium difficile* [@Freter1955;@Fekety1979;@Britton2012]. *C. difficile* infection (CDI) is responsible for most cases of antibiotic-associated colitis, a toxin-mediated diarrheal disease that has dramatically increased in prevalence over the last 10 years. There are an estimated 453,000 cases of CDI resulting in 29,000 deaths in the US annually [@Lessa2015]. Antibiotics are a major risk factor for CDI and are thought to increase susceptibility to CDI as they disrupt the gut bacterial community structure; however, it is still unclear what specific changes to the microbiota contribute to this susceptibility [@Antonopoulos2009;@Buffie2012]. While most classes of antibiotics have been associated with initial susceptibility to CDI, fluoroquinolones, clindamycin, and cephalosporins are linked to increased risk of recurrent or persistent infection [@Thomas2003;@Brown2013;@Bignardi1998]. This raises questions about the groups of bacteria that are differentially impacted by certain therapies and how these changes effect duration or severity of the infection.

Associations between the membership and functional capacity of the microbiota as measured by the metabolic output suggest that antibiotics increase susceptibility by altering the nutrient milieu in the gut to one that favors *C. difficile* metabolism [@Antunes2011;@Jump2014;@Theriot2014]. One hypothesis is that *C. difficile* colonization resistance is driven by competition for growth substrates by an intact community of metabolic specialists. This has been supported by animal model experiments over the past several decades [@Wilson1988;@Sambol2002;@Perez-Cobas2014]. This line of reasoning has been carried through to the downstream restoration of colonization resistance with the application of fecal microbiota transplant (FMT). Although an individual's microbiota may not return to its precise original state following FMT, it is hypothesized that the functional capacity of the new microbiota is able to outcompete *C. difficile* for resources and clear the infection [@Zaura2015;@Theriot2014]. 

Leveraging distinct antibiotic treatment regimens in a murine model of CDI [@Schubert2015], we and others have shown that *C. difficile* adapts its physiology to the distinct cecal microbiomes that resulted from exposure to antibiotics [@Schubert2015;@Jenior2017]. We went on to show that *C. difficile* appears to adapts portions of its metabolism to fit alternative nutrient niche landscapes presented. As the diet of the mice remained unchanged, changes in the cecal metabolome are likely driven by the intestinal microbiota. Although it has been established that *C. difficile* colonizes these communities effectively, it is unknown whether the differences in the metabolic activity of communities following antibiotic treatment are impacted by *C. difficile* colonization or if they correlate with prolonged infection. Historically, it has been difficult to ascribe specific metabolic contributions to individual taxa within the microbiota during perturbations, especially within the context of a host. To address this limited understanding, we employed an integrative untargeted metabolomic and metagenome-enabled metatransciptomic approach to investigate specific responses to infection of the gut microbiota in a murine model of CDI. This high-dimensional analysis allowed us to not only characterize the metabolic output of the community, but to also identify which subgroups of bacteria were differentially active during mock infection and CDI. Our results supported the hypothesis that CDI was indeed associated with altered community-level gene transcription and metabolomic profile of susceptible environments. This effect was significantly more pronounced in communities where *C. difficile* was able to maintain colonization. This work highlights the need for increased appreciation differential, combined effects of antibiotics and CDI on the gut microbiota in order to develop more successful targeted therapies that eliminate *C. difficile* colonization.

```{r startup, echo=F, message=F, warning=F, cache=F}

# Load dependencies
deps <- c('vegan', 'shape', 'plotrix', 'reshape2', 'GMD', 'randomForest', 'RColorBrewer', 'gplots','viridis', 'scales','Hmisc','VennDiagram')
for (dep in deps){
  if (dep %in% installed.packages()[,"Package"] == FALSE){
    install.packages(as.character(dep), quiet=TRUE);
  } 
  library(dep, verbose=FALSE, character.only=TRUE)
}
rm(dep, deps)

# Filter out columns that have values in at least 3 samples (ignores first column if needed)
filter_table <- function(data) {

  drop <- c()
  if (class(data[,1]) != 'character') {
    if (sum(data[,1] != 0) < 3) {
      drop <- c(drop, colnames(data)[1])
    }
  }
  
  for (index in 2:ncol(data)) {
    if (sum(data[,index] != 0) < 3) {
      drop <- c(drop, colnames(data)[index])
    }
  }
  
  filtered_data <- data[,!(colnames(data) %in% drop)]
  return(filtered_data)
}

# Neatly merge 2 matices with shared row names
clean_merge <- function(data_1, data_2){
  
  clean_merged <- merge(data_1, data_2, by='row.names', all.y=TRUE)
  rownames(clean_merged) <- clean_merged$Row.names
  clean_merged$Row.names <- NULL
  
  return(clean_merged)
}


# Significant feature filter with random forest (2 classes only)
susceptibilityRF <- function(feature_data){
  
  # Breiman (2001). Random Forests. Machine Learning.
  lvls <- as.vector(unique(feature_data$susceptibility))
  x <- round(length(rownames(feature_data[which(feature_data$susceptibility==lvls[1]),])) * 0.623)
  y <- round(length(rownames(feature_data[which(feature_data$susceptibility==lvls[2]),])) * 0.623)
  z <- max(c(round(x / y), round(y / x))) * 3
  nTrees <- round(ncol(feature_data) - 1) * z
  mTries <- round(sqrt(ncol(feature_data) - 1))
  rm(lvls, x, y, z)
  
  # Finish reformatting data
  training_data <- feature_data
  susceptibility <- training_data$susceptibility
  susceptibility <- as.factor(susceptibility)
  training_data$susceptibility <- NULL
  names(training_data) <- make.names(names(training_data))
  training_data <- data.matrix(training_data)
  training_data[is.na(training_data)] <- 0
  
  # Run random forest and get MDA values
  set.seed(906801)
  modelRF <- randomForest(susceptibility ~ ., data=training_data, 
                          importance=TRUE, replace=FALSE, err.rate=TRUE, 
                          ntree=nTrees, mtry=mTries)
  featRF <- importance(modelRF, type=1)
  featRF <- as.data.frame(featRF)
  featRF$features <- rownames(featRF)
  rm(modelRF)
  
  # Filter to significant features (Strobl 2002) and sort
  sigFeatRF <- as.data.frame(subset(featRF, featRF$MeanDecreaseAccuracy > (abs(min(featRF$MeanDecreaseAccuracy)))))
  
  # Subset trainng data to significant features
  finalFeat <- training_data[,which(rownames(sigFeatRF) %in% colnames(training_data))]
  rm(training_data, sigFeatRF)
  return(finalFeat)
}

# Significant feature filter with random forest (2 classes only)
infectionRF <- function(feature_data){
  
  # Breiman (2001). Random Forests. Machine Learning.
  lvls <- as.vector(unique(feature_data$infection))
  x <- round(length(rownames(feature_data[which(feature_data$infection==lvls[1]),])) * 0.623)
  y <- round(length(rownames(feature_data[which(feature_data$infection==lvls[2]),])) * 0.623)
  z <- max(c(round(x / y), round(y / x))) * 3
  nTrees <- round(ncol(feature_data) - 1) * z
  mTries <- round(sqrt(ncol(feature_data) - 1))
  rm(lvls, x, y, z)
  
  # Finish reformatting data
  training_data <- feature_data
  infection <- training_data$infection
  infection <- as.factor(infection)
  training_data$infection <- NULL
  names(training_data) <- make.names(names(training_data))
  training_data <- data.matrix(training_data)
  training_data[is.na(training_data)] <- 0
  
  # Run random forest and get MDA values
  set.seed(906801)
  modelRF <- randomForest(infection ~ ., data=training_data, 
                          importance=TRUE, replace=FALSE, err.rate=TRUE, 
                          ntree=nTrees, mtry=mTries)
  featRF <- importance(modelRF, type=1)
  featRF <- as.data.frame(featRF)
  featRF$features <- rownames(featRF)
  rm(modelRF)
  
  # Filter to significant features (Strobl 2002) and sort
  sigFeatRF <- as.data.frame(subset(featRF, featRF$MeanDecreaseAccuracy > (abs(min(featRF$MeanDecreaseAccuracy)))))
  sigFeatRF <- sigFeatRF[order(-sigFeatRF$MeanDecreaseAccuracy),]
  return(sigFeatRF)
}

# Significant feature filter with random forest (2 or more classes)
abxRF <- function(feature_data){

  mTries <- round(sqrt(ncol(feature_data) - 1))

  # Finish reformatting data
  training_data <- feature_data
  abx <- training_data$abx
  abx <- as.factor(abx)
  training_data$abx <- NULL
  names(training_data) <- make.names(names(training_data))
  training_data <- data.matrix(training_data)
  training_data[is.na(training_data)] <- 0
  
  # Run random forest and get MDA values
  set.seed(906801)
  modelRF <- randomForest(abx ~ ., data=training_data, 
                          importance=TRUE, replace=FALSE, err.rate=TRUE, mtry=mTries)
  featRF <- importance(modelRF, type=1)
  featRF <- as.data.frame(featRF)
  featRF$features <- rownames(featRF)
  rm(modelRF)
  
  # Filter to significant features (Strobl 2002) and sort
  sigFeatRF <- as.data.frame(subset(featRF, featRF$MeanDecreaseAccuracy > (abs(min(featRF$MeanDecreaseAccuracy)))))
  
  # Subset trainng data to significant features
  finalFeat <- training_data[,which(rownames(sigFeatRF) %in% colnames(training_data))]
  rm(training_data, sigFeatRF)
  return(finalFeat)
}

# Calculates distance of a point from x=y in 2-d space
dist_xy <- function(x) {
  v1 <- c(20,20) - x
  v2 <- c(0,0) - c(20,20)
  m <- cbind(v1, v2)
  distance <- abs(det(m)) / sqrt(sum(v1 * v1))
  return(distance)
} 

```

**Results**

```{r figure_1, echo=F, warning=F, message=F, results='hide', cache=F}

# Load in data
wetlab <- read.delim('data/wetlab_assays.tsv', sep='\t', header=T, row.names=1)

# Format wetlab assay data
wetlab <- subset(wetlab, infection == '630') # Remove uninfected controls
wetlab <- subset(wetlab, treatment != 'germfree') # Remove germfree
wetlab$infection <- NULL
wetlab$cfu_vegetative <- as.numeric(wetlab$cfu_vegetative)
wetlab$cfu_vegetative[wetlab$cfu_vegetative == 0] <- 100
wetlab$cfu_spore <- as.numeric(wetlab$cfu_spore)
wetlab$cfu_spore[wetlab$cfu_spore == 0] <- 100
wetlab$toxin_titer <- as.numeric(wetlab$toxin_titer)

# Separate datasets
strep_veg <- subset(wetlab, treatment == 'streptomycin')$cfu_vegetative
strep_spore <- subset(wetlab, treatment == 'streptomycin')$cfu_spore
strep_tox <- subset(wetlab, treatment == 'streptomycin')$toxin_titer
cef_veg <- subset(wetlab, treatment == 'cefoperazone')$cfu_vegetative
cef_spore <- subset(wetlab, treatment == 'cefoperazone')$cfu_spore
cef_tox <- subset(wetlab, treatment == 'cefoperazone')$toxin_titer
clinda_veg <- subset(wetlab, treatment == 'clindamycin')$cfu_vegetative
clinda_spore <- subset(wetlab, treatment == 'clindamycin')$cfu_spore
clinda_tox <- subset(wetlab, treatment == 'clindamycin')$toxin_titer
noabx_veg <- subset(wetlab, treatment == 'conventional')$cfu_vegetative
noabx_spore <- subset(wetlab, treatment == 'conventional')$cfu_spore
noabx_tox <- subset(wetlab, treatment == 'conventional')$toxin_titer
rm(wetlab)

# Calculate differences
# Vegetative
veg_pval <- p.adjust(c(wilcox.test(noabx_veg, strep_veg, exact=F)$p.value,
                       wilcox.test(noabx_veg, cef_veg, exact=F)$p.value,
                       wilcox.test(noabx_veg, clinda_veg, exact=F)$p.value), method='BH')
# Spores
spore_pval <- p.adjust(c(wilcox.test(noabx_spore, strep_spore, exact=F)$p.value,
                       wilcox.test(noabx_spore, cef_spore, exact=F)$p.value,
                       wilcox.test(noabx_spore, clinda_spore, exact=F)$p.value), method='BH')
# Toxin titer
tox_pval <- p.adjust(c(wilcox.test(noabx_tox, strep_tox, exact=F)$p.value,
                       wilcox.test(noabx_tox, cef_tox, exact=F)$p.value,
                       wilcox.test(noabx_tox, clinda_tox, exact=F)$p.value), method='BH')
p_values <- c(veg_pval, spore_pval, tox_pval)
wetlab_pval <- round(max(p_values), 3)
rm(p_values, veg_pval, spore_pval, tox_pval)

```


```{r figure_s1, echo=F, warning=F, message=F, results='hide', cache=F}

# Read in data
# 16S data
shared_otu <- read.delim('data/16S_analysis/all_treatments.0.03.unique_list.conventional.shared', sep='\t', header=TRUE, row.names=2)
# Metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=T, row.names=1)

# Format data
# Metadata
metadata <- metadata[!rownames(metadata) %in% c('CefC5M2'), ] 
metadata$type <- NULL
metadata$cage <- NULL
metadata$mouse <- NULL
metadata$gender <- NULL
# 16S data
shared_otu$label <- NULL
shared_otu$numOtus <- NULL
shared_otu <- shared_otu[!rownames(shared_otu) %in% c('CefC5M2'), ]

# Remove residual C. difficile OTUs
cdiff_otu <- shared_otu[,(names(shared_otu) %in% c('Otu0004','Otu0308'))]
cdiff_otu <- clean_merge(metadata, cdiff_otu)
cdiff_otu <- subset(cdiff_otu, infection == '630')
cdiff_otu <- as.numeric(sum(sum(cdiff_otu$Otu0004) + sum(cdiff_otu$Otu0308)))
shared_otu <- shared_otu[,!(names(shared_otu) %in% c('Otu0004','Otu0308'))]
other_otu <- clean_merge(metadata, shared_otu)
other_otu <- subset(other_otu, infection == '630')
other_otu$abx <- NULL
other_otu$infection <- NULL
other_otu$susceptibility <- NULL
other_otu <- as.numeric(sum(colSums(other_otu)))
cdiff_percent <- as.character(round((cdiff_otu / other_otu) * 100, 2)) # Calculate % removed reads
rm(cdiff_otu, other_otu)

# Subsample shared file
sub_sample <- ceiling(min(rowSums(shared_otu)) * 0.9) # calculate rarefaction level
shared_otu <- rrarefy(shared_otu, sub_sample) # subsample shared file
shared_otu <- filter_table(shared_otu)
rm(sub_sample)

# Calculate axes
# Community structure - all
otu_nmds <- metaMDS(shared_otu, k=2, trymax=100, distance='bray')$points
otu_nmds <- clean_merge(metadata, otu_nmds)

# All abx
abx_otu <- shared_otu[!rownames(shared_otu) %in% c('ConvC1M1','ConvC1M2','ConvC1M3','ConvC1M4',
                                                   'ConvC2M1','ConvC2M2','ConvC2M3','ConvC2M4','ConvC2M5'), ]
# Individual abx
cef_otu <- shared_otu[rownames(shared_otu) %in% c('CefC1M1','CefC1M2','CefC1M3',
                                                  'CefC2M1','CefC2M2','CefC2M3',
                                                  'CefC3M1','CefC3M2','CefC3M3',
                                                  'CefC4M1','CefC4M2','CefC4M3',
                                                  'CefC5M1','CefC5M2','CefC5M3',
                                                  'CefC6M1','CefC6M2','CefC6M3'), ]
clinda_otu <- shared_otu[rownames(shared_otu) %in% c('ClindaC1M1','ClindaC1M2','ClindaC1M3',
                                                     'ClindaC2M1','ClindaC2M2','ClindaC2M3',
                                                     'ClindaC3M1','ClindaC3M2','ClindaC3M3',
                                                     'ClindaC4M1','ClindaC4M2','ClindaC4M3',
                                                     'ClindaC5M1','ClindaC5M2','ClindaC5M3',
                                                     'ClindaC6M1','ClindaC6M2','ClindaC6M3'), ]
strep_otu <- shared_otu[rownames(shared_otu) %in% c('StrepC1M1','StrepC1M2','StrepC1M3',
                                                    'StrepC2M1','StrepC2M2','StrepC2M3',
                                                    'StrepC3M1','StrepC3M2','StrepC3M3',
                                                    'StrepC4M1','StrepC4M2','StrepC4M3',
                                                    'StrepC5M1','StrepC5M2','StrepC5M3',
                                                    'StrepC6M1','StrepC6M2','StrepC6M3'), ]

# Calculate inv-Simpson diversity (alpha)
cef_infected_diversity <- diversity(cef_otu[rownames(cef_otu) %in% c('CefC1M1','CefC1M2','CefC1M3',
                                                    'CefC2M1','CefC2M2','CefC2M3',
                                                    'CefC3M1','CefC3M2','CefC3M3'), ], 'invsimpson')
cef_mock_diversity <- diversity(cef_otu[rownames(cef_otu) %in% c('CefC4M1','CefC4M2','CefC4M3',
                                                'CefC5M1','CefC5M2','CefC5M3',
                                                'CefC6M1','CefC6M2','CefC6M3'), ], 'invsimpson')
clinda_infected_diversity <- diversity(clinda_otu[rownames(clinda_otu) %in% c('ClindaC1M1','ClindaC1M2','ClindaC1M3',
                                                          'ClindaC2M1','ClindaC2M2','ClindaC2M3',
                                                          'ClindaC3M1','ClindaC3M2','ClindaC3M3'), ], 'invsimpson')
clinda_mock_diversity <- diversity(clinda_otu[rownames(clinda_otu) %in% c('ClindaC4M1','ClindaC4M2','ClindaC4M3',
                                                      'ClindaC5M1','ClindaC5M2','ClindaC5M3',
                                                      'ClindaC6M1','ClindaC6M2','ClindaC6M3'), ], 'invsimpson')
strep_infected_diversity <- diversity(strep_otu[rownames(strep_otu) %in% c('StrepC1M1','StrepC1M2','StrepC1M3',
                                                        'StrepC2M1','StrepC2M2','StrepC2M3',
                                                        'StrepC3M1','StrepC3M2','StrepC3M3'), ], 'invsimpson')
strep_mock_diversity <- diversity(strep_otu[rownames(strep_otu) %in% c('StrepC4M1','StrepC4M2','StrepC4M3',
                                                    'StrepC5M1','StrepC5M2','StrepC5M3',
                                                    'StrepC6M1','StrepC6M2','StrepC6M3'), ], 'invsimpson')
noabx_diversity <- diversity(shared_otu[rownames(shared_otu) %in% c('ConvC1M1','ConvC1M2','ConvC1M3','ConvC1M4',
                                                    'ConvC2M1','ConvC2M2','ConvC2M3','ConvC2M4','ConvC2M5'), ], 
                             'invsimpson')

# Test differences in diversity
# Untreated vs abx
noabx_div_pval <- p.adjust(c(wilcox.test(noabx_diversity, cef_infected_diversity, exact=F)$p.value,
                             wilcox.test(noabx_diversity, cef_mock_diversity, exact=F)$p.value,
                             wilcox.test(noabx_diversity, clinda_infected_diversity, exact=F)$p.value,
                             wilcox.test(noabx_diversity, clinda_mock_diversity, exact=F)$p.value,
                             wilcox.test(noabx_diversity, strep_infected_diversity, exact=F)$p.value,
                             wilcox.test(noabx_diversity, strep_mock_diversity, exact=F)$p.value), method='BH')
noabx_div_pval <- as.character(max(round(noabx_div_pval, 3)))
strep_div_pval <- round(wilcox.test(strep_infected_diversity, strep_mock_diversity, exact=F)$p.value, 3)
cef_div_pval <- round(wilcox.test(cef_infected_diversity, cef_mock_diversity, exact=F)$p.value, 3)
clinda_div_pval <- round(wilcox.test(clinda_infected_diversity, clinda_mock_diversity, exact=F)$p.value, 3)
abx_div_pval <- as.character(max(c(strep_div_pval,cef_div_pval,clinda_div_pval)))
rm(strep_infected_diversity, strep_mock_diversity, cef_infected_diversity, cef_mock_diversity, 
   clinda_infected_diversity, clinda_mock_diversity, noabx_diversity,
   strep_div_pval, cef_div_pval, clinda_div_pval)

# Calculate axes and merge with metadata
abx_otu_nmds <- metaMDS(abx_otu, k=2, trymax=100)$points
abx_otu_nmds <- clean_merge(metadata, abx_otu_nmds)
cef_otu_nmds <- metaMDS(cef_otu, k=2, trymax=100)$points
cef_otu_nmds <- clean_merge(metadata, cef_otu_nmds)
clinda_otu_nmds <- metaMDS(clinda_otu, k=2, trymax=100)$points
clinda_otu_nmds <- clean_merge(metadata, clinda_otu_nmds)
strep_otu_nmds <- metaMDS(strep_otu, k=2, trymax=100)$points
strep_otu_nmds <- clean_merge(metadata, strep_otu_nmds)
rm(metadata)

# Calculate significant differences
otu_p <- as.character(round(anosim(shared_otu, otu_nmds$susceptibility, permutations=999, distance='bray')$signif, 3))
otu_r <- as.character(round(abs(anosim(shared_otu, otu_nmds$susceptibility, permutations=999, distance='bray')$statistic), 3))
abx_otu_p <- as.character(round(anosim(abx_otu, abx_otu_nmds$abx, permutations=999, distance='bray')$signif, 3))
abx_otu_r <- as.character(round(abs(anosim(abx_otu, abx_otu_nmds$abx, permutations=999, distance='bray')$statistic), 3))
cef_otu_p <- as.character(round(anosim(cef_otu, cef_otu_nmds$infection, permutations=999, distance='bray')$signif, 3))
cef_otu_r <- as.character(round(abs(anosim(cef_otu, cef_otu_nmds$infection, permutations=999, distance='bray')$statistic), 3))
clinda_otu_p <- as.character(round(anosim(clinda_otu, clinda_otu_nmds$infection, permutations=999, distance='bray')$signif, 3))
clinda_otu_r <- as.character(round(abs(anosim(clinda_otu, clinda_otu_nmds$infection, permutations=999, distance='bray')$statistic), 3))
strep_otu_p <- as.character(round(anosim(strep_otu, strep_otu_nmds$infection, permutations=999, distance='bray')$signif, 3))
strep_otu_r <- as.character(round(abs(anosim(strep_otu, strep_otu_nmds$infection, permutations=999, distance='bray')$statistic), 3))
rm(abx_otu, cef_otu, clinda_otu, strep_otu, shared_otu, 
    otu_nmds, abx_otu_nmds, cef_otu_nmds, clinda_otu_nmds, strep_otu_nmds)

```

**Distinct antibiotic pretreatments are associated with different patterns of clearance and virulence factor expression.**
In previous work from our laboratory it was found that when conventionally-reared SPF mice were pretreated with one of three different antibiotics (streptomycin, cefoperazone, and clindamycin; Table S1) to sensitize the animals to *C. difficile* colonization, each pretreatment was associated with altered patterns of *C. difficile* virulence factor expression (Fig. 1A) [@Jenior2017]. Briefly, these drugs were chosen for not only the ability to to reduce *C. difficile* colonization resistance in a mouse model [@Schubert2015], but also for distinct and significant impacts on the structure and diversity of the cecal microbiota (all *p* < `r noabx_div_pval`; Fig. S1A), as well as differential patterns of *C. difficile* str. 630 clearance following initial infection as quantified from stool (Fig. 1B). In each antibiotic pretreatment model, we observed equally high levels of *C. difficile* colonization on the day after infection. Out of the three pretreatment regimes however, *C. difficile* fell below the limit of detection in only clindamycin-pretreated mice over the subsequent 8 days while the other two pretreatments remained highly colonized. We hypothesized that this occurred in the clindamycin-pretreated mice because the resultant intestinal community occupied niche space with greater overlap to that of *C. difficile*. 

Earlier publications have demonstrated that *C. difficile* virulence factor expression is regulated by availability of certain nutrients in the environment [@Bouillaut2015], so we first sought to measure spore and toxin production. We chose to focus all downstream experimentation on 18-hours post-infection to the assess behavior of *C. difficile* directly prior to the reduction in detectable *C. difficile*. This end point corresponded with a previous study where *C. difficile* reached maximum cecal vegetative cell load with few detectable spores [@Koenigsknecht2015]. We also elected to examine cecal content because it was more likely to be a site of active bacterial metabolism compared to stool and would allow for an assessment of functional differences in the microbiota. At 18 hours after infection, there were no significant differences in the number of vegetative cells between any antibiotic-pretreatment group. All susceptible mice were colonized with ~1x10^8^ vegetative colony forming units (cfu) per gram of cecal content and untreated mice maintained *C. difficile* colonization resistance (Fig. 1C). We also measured both spore production and toxin activity as activation of both processes has been linked to environmental concentrations of specific growth nutrients [@Bouillaut2015]. Despite having similar vegetative *C. difficile* load, varying levels of both cfu due to spores and toxin titer were observed across each of the antibiotic pretreatments prior to the onset of detectable inflammation in the epithelium [@Koenigsknecht2015]. These results suggested that despite high initial *C. difficile* colonization, the microbiomes across pretreatments may vary in available nutrients or profiles of competitors for those niches. In order to maintain consistency for our investigation of these communities, we performed all subsequent analyses utilizing cecal content collected during the previously described experiments.

***C. difficile* colonization corresponded with significantly altered cecal community structures.**
It has not yet been established whether *C. difficile* colonization impacts the structure of the gut microbiota during infection in susceptible mice. To evaluate changes in bacterial population abundances in response to antibiotic perturbation and *C. difficile* colonization, we sequenced the V4 region of the 16S rRNA gene from the cecal content of both mock and *C. difficile*-infected mice across antibiotic pretreatment models. To focus our analysis specifically on the surrounding bacterial communities we removed all *C. difficile* 16S rRNA gene sequences, which represented `r cdiff_percent`% of sequencing reads across infection groups. We confirmed that each antibiotic pretreatment significantly impacted both cecal community structure and diversity compared to untreated control mice (*p* < `r otu_p`, Fig. S1A & S1B). We then established that each antibiotic pretreatment induced distinct and significant shifts in the community structure (*p* < `r abx_otu_p`, Fig. S1C). The composition of streptomycin-pretreated communities was more variable between cages, but was generally enriched for members of phylum *Bacteroidetes* (Fig. 2A). Cefoperazone and clindamycin-pretreated cecal communities were consistently dominated by members of the *Lactobacillaceae* and *Enterobacteriaceae* families, respectively (Fig. 2A). Despite variation in the community structures generated by streptomycin pretreatment, all communities were colonized by *C. difficile* to the same level (Fig. 1B, 1C, & S1).

Next, we measured the effect that *C. difficile* colonization had on overall community structure and composition within each of the antibiotic groups individually. We found that the structure of both streptomycin and clindamycin-pretreated infected communities were significantly different from their corresponding uninfected controls (streptomycin *p* = `r strep_otu_p`, Fig. S1D; clindamycin *p* = `r clinda_otu_p`, Fig. S1F). Cefoperazone-pretreated communities did not significantly differ from their control group (Fig. S1E). Utilizing LEfSe differential abundant feature detection [@Segata2011], we identified 16 OTUs that discriminated between infected and uninfected communities in at least one antibiotic pretreatment (Fig. 2BCD); however, these OTUs were generally near the limit of detection and very few were present across multiple groups tested. This suggested that CDI susceptible communities were only modestly impacted by the presence of *C. difficile*.

```{r figure_3, echo=F, warning=F, message=F, results='hide', cache=F}

# Metabolomes
metabolome <- read.delim('data/metabolome/scaled_intensities.log10.tsv', sep='\t', header=TRUE)

# Metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=T, row.names=1)

# Format data
# Metadata
metadata$type <- NULL
metadata$cage <- NULL
metadata$mouse <- NULL
metadata$gender <- NULL
# Metabolomes
metabolome$BIOCHEMICAL <- gsub('_', ' ', metabolome$BIOCHEMICAL)
rownames(metabolome) <- metabolome$BIOCHEMICAL
metabolome$BIOCHEMICAL <- NULL
metabolome$PUBCHEM <- NULL
metabolome$KEGG <- NULL
metabolome$SUB_PATHWAY <- NULL
metabolome$SUPER_PATHWAY <- NULL
metabolome <- as.data.frame(t(metabolome))

# Stats
# Prep data
metabolome <- clean_merge(metadata, metabolome)
metabolome <- subset(metabolome, abx != 'germfree')
metabolome <- subset(metabolome, infection == 'mock')
abx_metabolome <- subset(metabolome, abx != 'none')
abx_metabolome$infection <- NULL
abx_metabolome$susceptibility <- NULL
metabolome$abx <- NULL
metabolome$infection <- NULL

# Calculate significant differences
noabx_sus_p <- as.character(round(adonis(metabolome[,2:ncol(metabolome)]~factor(metabolome$susceptibility),
                                     data=metabolome, permutations=10000, method='jaccard')$aov.tab[[6]][1], 3))
if (noabx_sus_p == '0') {
noabx_sus_p = '0.001'
}

abx_sus_p <- as.character(round(adonis(abx_metabolome[,2:ncol(abx_metabolome)]~factor(abx_metabolome$abx),
                                   data=abx_metabolome, permutations=10000, method='jaccard')$aov.tab[[6]][1], 3))
metabolome$susceptibility <- NULL
abx_metabolome$abx <-  NULL

# Feature selection
# Separate groups
metabolome <- clean_merge(metadata, metabolome)
abx_metabolome <- subset(metabolome, abx != 'none')
abx_metabolome$infection <- NULL
abx_metabolome$susceptibility <- NULL
abx_metabolome$abx <- factor(abx_metabolome$abx)
metabolome$infection <- NULL
metabolome$abx <- NULL
metabolome$abx <- factor(metabolome$susceptibility)

# Random Forest
all_rf <- susceptibilityRF(metabolome)
abx_rf <- abxRF(abx_metabolome)
rm(metabolome, abx_metabolome)

# Subset concentrations
all_rf <- clean_merge(metadata, all_rf)
res_metabolome <- subset(all_rf, susceptibility == 'resistant')
res_metabolome$susceptibility <- NULL
res_metabolome$abx <- NULL
res_metabolome$infection <- NULL
sus_metabolome <- subset(all_rf, susceptibility == 'susceptible')
sus_metabolome$susceptibility <- NULL
sus_metabolome$abx <- NULL
sus_metabolome$infection <- NULL
abx_rf <- clean_merge(metadata, abx_rf)
cef_abx_metabolome <- subset(abx_rf, abx == 'cefoperazone')
cef_abx_metabolome$abx <- NULL
cef_abx_metabolome$infection <- NULL
cef_abx_metabolome$susceptibility <- NULL
clinda_abx_metabolome <- subset(abx_rf, abx == 'clindamycin')
clinda_abx_metabolome$abx <- NULL
clinda_abx_metabolome$infection <- NULL
clinda_abx_metabolome$susceptibility <- NULL
strep_abx_metabolome <- subset(abx_rf, abx == 'streptomycin')
strep_abx_metabolome$abx <- NULL
strep_abx_metabolome$infection <- NULL
strep_abx_metabolome$susceptibility <- NULL
rm(all_rf, abx_rf, metadata)

# Find significant differences
resistant_pvalues <- c()
for (i in 1:ncol(res_metabolome)){resistant_pvalues[i] <- wilcox.test(res_metabolome[,i], sus_metabolome[,i], exact=FALSE)$p.value}
resistant_pvalues <- as.character(max(round(p.adjust(resistant_pvalues, method='BH'),3)))
rm(res_metabolome, sus_metabolome)

abx_pvalues1 <- c()
for (i in 1:ncol(cef_abx_metabolome)){abx_pvalues1[i] <- wilcox.test(cef_abx_metabolome[,i], clinda_abx_metabolome[,i], exact=FALSE)$p.value}
abx_pvalues2 <- c()
for (i in 1:ncol(cef_abx_metabolome)){abx_pvalues2[i] <- wilcox.test(cef_abx_metabolome[,i], strep_abx_metabolome[,i], exact=FALSE)$p.value}
abx_pvalues3 <- c()
for (i in 1:ncol(clinda_abx_metabolome)){abx_pvalues3[i] <- wilcox.test(clinda_abx_metabolome[,i], strep_abx_metabolome[,i], exact=FALSE)$p.value}
abx_pvalues1 <- as.character(max(round(p.adjust(abx_pvalues1, method='BH'),3)))
abx_pvalues2 <- as.character(max(round(p.adjust(abx_pvalues2, method='BH'),3)))
abx_pvalues3 <- as.character(max(round(p.adjust(abx_pvalues3, method='BH'),3)))
rm(strep_abx_metabolome, cef_abx_metabolome, clinda_abx_metabolome)

```

**Distinct antibiotic classes lead to enrichment of alternative *C. difficile* nutrient niches.**
Pretreatment with antibiotics not only alters the structure of the resident microbiota, but also has a dramatic impact on the intestinal metabolome [@Antunes2011; @Jump2014; @Theriot2014]. To test the metabolic features of the altered communities, we performed untargeted metabolomic analysis on the cecal contents that were also utilized in the 16S rRNA gene sequencing. A total of 727 distinct metabolites were identified. First, we characterized the differences between the metabolomes of the mock-infected communities to measure the impact of antibiotic pretreatment that generated *C. difficile*-susceptible conditions (Fig. 3). All of the antibiotic pretreatments significantly altered the cecal metabolome compared to untreated, *C. difficile*-resistant mice as quantified by Bray-Curtis dissimilarity (*p* < `r noabx_sus_p`; Fig. 3A). As with the differences between resistant and susceptible states, the patterns of specific metabolite concentrations diverged depending on the antibiotic pretreatment (Fig. 3). These results demonstrated that each pretreatment lead to distinct susceptible metabolomic environments.

We were interested in identifying those metabolites that best indicate the changes induced by antibiotic pretreatment, and potentially contributing sensitivity to *C. difficile* colonization. Standard analyses revealed large numbers of individual metabolites with significant differences between groups, so we employed an implementation of the Random Forest machine-learning algorithm [@Calle2011] to identify the smallest subset of metabolites most able to differentiate states. This technique revealed 5 metabolites that together correctly classified all metabolomes as either resistant or susceptible (Fig. 3C). The first 2 are significantly elevated in susceptible metabolomes (*p* < `resistant_pvalues`). Acetlyproline has been previously shown to be a growth substrate of *C. difficile* through Stickland fermentation [@Fonknechten2010], and may be indicate a universal marker of CDI susceptibility. Interestingly nudifloramide, or N-methyl-2-pyridone-5-carboxamide, is an end product of NAD degradation which has been associated with multiple unrelated intestinal disease states [@Rutkowski2003;@Mayneris-Perxachs2016] and may be a general indicator of dysbiosis in the GI tract. The remaining 3 (sebacate, hyodeoxycholate, and murideoxycholate) are all related to secondary bile acids [@Semba2017] indicating a possible decrease in germination inhibitors across susceptible animals. These data support the hypothesis that amino acid catabolism is disrupted at a community-level following antibiotic pretreatment and may suggest open nutrient niches for *C. difficile*. 

We then applied the same approach to distinguish antibiotic pretreatment groups from each other (Fig. 3D). This featured 5 additional metabolites able to correctly classify susceptible environments, and highlighted the metabolic diversity inherent in each of these microbiomes. As the distinct antibiotics permitted survival of separate bacterial groups (Fig. 2A), likely with varied nutrient catabolism strategies, that each may compete with *C. difficile* along different axes. Further analysis of metabolites which have been shown to be deeply connected to the life cycle of *C. difficile* [@Theriot2014;@Sorg2010;@Buffie2014], also indicated high degrees of variablility between these environments. Together, our results supported that each susceptible environment was distinguishable from the others, with its own subset of enriched *C. difficile* growth substrates (Fig. 2B-C). This could be an indication that particular competitors were eliminated during antibiotic pretreatment, or those community members normally responsible for the consumption of these metabolites have altered their metabolic program to exploit alternative nutrient sources.


```{r figure_s3, echo=F, warning=F, message=F, results='hide', cache=F}

# Read in data
# Metabolomes
metabolome <- read.delim('data/metabolome/scaled_intensities.log10.tsv', sep='\t', header=TRUE)
# Metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=T, row.names=1)

# Format data
# Metadata
metadata$type <- NULL
metadata$cage <- NULL
metadata$mouse <- NULL
metadata$gender <- NULL
# Metabolomes
metabolome <- metabolome[,!colnames(metabolome) %in% c('GfC1M1','GfC1M2','GfC1M3', 
                                                       'GfC2M1','GfC2M2','GfC2M3', 
                                                       'GfC3M1','GfC3M2','GfC3M3', 
                                                       'GfC4M1','GfC4M2','GfC4M3', 
                                                       'GfC5M1','GfC5M2','GfC5M3', 
                                                       'GfC6M1','GfC6M2','GfC6M3')] # Remove germfree
rownames(metabolome) <- metabolome$BIOCHEMICAL
metabolome$BIOCHEMICAL <- NULL
metabolome$PUBCHEM <- NULL
metabolome$KEGG <- NULL
metabolome$SUB_PATHWAY <- NULL
metabolome$SUPER_PATHWAY <- NULL
metabolome <- as.data.frame(t(metabolome))

# Prep data
metabolome <- clean_merge(metadata, metabolome)
infection_metabolome <- subset(metabolome, abx != 'none')
infection_metabolome$abx <- NULL
infection_metabolome$susceptibility <- NULL
rm(metadata, metabolome)

# Calculate significant differences
inf_p <- as.character(round(adonis(infection_metabolome[,2:ncol(infection_metabolome)]~factor(infection_metabolome$infection), data=infection_metabolome, permutations=999, method='jaccard')$aov.tab[[6]][1], 3))

# Random Forest and get top hits
inf_rf <- infectionRF(infection_metabolome)
inf_rf <- inf_rf[1:10,]

# Subset concentrations
colnames(infection_metabolome) <- make.names(colnames(infection_metabolome))
inf_infection_metabolome <- subset(infection_metabolome, infection == '630')[, inf_rf$feature]
inf_infection_metabolome$infection <- NULL
mock_infection_metabolome <- subset(infection_metabolome, infection == 'mock')[, inf_rf$feature]
mock_infection_metabolome$infection <- NULL
rm(infection_metabolome, inf_rf)

# Find significant differences
infection_pvalues <- c()
for (i in 1:ncol(inf_infection_metabolome)){infection_pvalues[i] <- wilcox.test(inf_infection_metabolome[,i], mock_infection_metabolome[,i], exact=FALSE)$p.value}
infection_metabolites <- colnames(inf_infection_metabolome)
infection_pvalues <- as.character(round(max(p.adjust(infection_pvalues, method='BH')),3))
infection_pvalues
if (infection_pvalues == '0') {
infection_pvalues = '0.001'
}
rm(inf_infection_metabolome)

```

```{r figure_4, echo=F, warning=F, message=F, results='hide', cache=F}

# Read in data
# Metabolomes
metabolome <- read.delim('data/metabolome/scaled_intensities.log10.tsv', sep='\t', header=TRUE)
aminovalerate <- read.delim('exploratory/aminovalerate.tsv', sep='\t', header=T)
# Metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=T, row.names=1)

# Format data
# Metadata
metadata$type <- NULL
metadata$cage <- NULL
metadata$mouse <- NULL
metadata$gender <- NULL
# Metabolomes
rownames(metabolome) <- metabolome$BIOCHEMICAL
metabolome$BIOCHEMICAL <- NULL
metabolome_metadata <- metabolome[,c(1:4)]
metabolome$PUBCHEM <- NULL
metabolome$KEGG <- NULL
metabolome$SUB_PATHWAY <- NULL
metabolome$SUPER_PATHWAY <- NULL
metabolome <- as.data.frame(t(metabolome))
aminovalerate <- subset(aminovalerate, abx != 'germfree')

# Prep data
metabolome <- clean_merge(metadata, metabolome)
metabolome <- subset(metabolome, abx != 'germfree')
cef_metabolome <- subset(metabolome, abx == 'cefoperazone')
cef_metabolome$abx <- NULL
cef_metabolome$susceptibility <- NULL
clinda_metabolome <- subset(metabolome, abx == 'clindamycin')
clinda_metabolome$abx <- NULL
clinda_metabolome$susceptibility <- NULL
strep_metabolome <- subset(metabolome, abx == 'streptomycin')
strep_metabolome$abx <- NULL
strep_metabolome$susceptibility <- NULL

# Calculate significant differences
strep_inf_p <- as.character(round(adonis(strep_metabolome[,2:ncol(strep_metabolome)]~factor(strep_metabolome$infection), data=strep_metabolome, permutations=999, method='jaccard')$aov.tab[[6]][1], 3))
strep_metabolome$infection <- factor(strep_metabolome$infection)
cef_inf_p <- as.character(round(adonis(cef_metabolome[,2:ncol(cef_metabolome)]~factor(cef_metabolome$infection), data=cef_metabolome,permutations=999, method='jaccard')$aov.tab[[6]][1], 3))
cef_metabolome$infection <- factor(cef_metabolome$infection)
clinda_inf_p <- as.character(round(adonis(clinda_metabolome[,2:ncol(clinda_metabolome)]~factor(clinda_metabolome$infection),data=clinda_metabolome, permutations=999, method='jaccard')$aov.tab[[6]][1], 3))
clinda_metabolome$infection <- factor(clinda_metabolome$infection)
rm(metadata, metabolome)

# Aminovalerate data
av_noabx <- subset(aminovalerate, abx == 'none')$aminovalerate
av_strep_inf <- subset(aminovalerate, abx == 'streptomycin' & infection == 'infected')$aminovalerate
av_strep_mock <- subset(aminovalerate, abx == 'streptomycin' & infection == 'mock')$aminovalerate
av_cef_inf <- subset(aminovalerate, abx == 'cefoperazone' & infection == 'infected')$aminovalerate
av_cef_mock <- subset(aminovalerate, abx == 'cefoperazone' & infection == 'mock')$aminovalerate
av_clinda_inf <- subset(aminovalerate, abx == 'clindamycin' & infection == 'infected')$aminovalerate
av_clinda_mock <- subset(aminovalerate, abx == 'clindamycin' & infection == 'mock')$aminovalerate
rm(aminovalerate)

# Stats
av_noabx_pval <- c(wilcox.test(av_noabx, av_strep_inf, exact=FALSE)$p.value,
                   wilcox.test(av_noabx, av_strep_mock, exact=FALSE)$p.value,
                   wilcox.test(av_noabx, av_cef_inf, exact=FALSE)$p.value,
                   wilcox.test(av_noabx, av_cef_mock, exact=FALSE)$p.value,
                   wilcox.test(av_noabx, av_clinda_inf, exact=FALSE)$p.value,
                   wilcox.test(av_noabx, av_clinda_mock, exact=FALSE)$p.value)
av_noabx_pval <- as.character(round(p.adjust(av_noabx_pval, method='BH'), 3))
rm(av_noabx)
av_strep_pval <- as.character(round(wilcox.test(av_strep_inf, av_strep_mock, exact=FALSE)$p.value, 3))
rm(av_strep_inf, av_strep_mock)
av_cef_pval <- as.character(round(wilcox.test(av_cef_inf, av_cef_mock, exact=FALSE)$p.value, 3))
rm(av_cef_inf, av_cef_mock)
av_clinda_pval <- as.character(round(wilcox.test(av_clinda_inf, av_clinda_mock, exact=FALSE)$p.value, 3))
rm(av_clinda_inf, av_clinda_mock)

# Random Forest
cef_rf <- infectionRF(cef_metabolome)
clinda_rf <- infectionRF(clinda_metabolome)
strep_rf <- infectionRF(strep_metabolome)

# Sort and subset top hits
cef_rf <- cef_rf[1:5,]
clinda_rf <- clinda_rf[1:5,]
strep_rf <- strep_rf[1:5,]

# Subset concentrations
colnames(cef_metabolome) <- make.names(colnames(cef_metabolome))
inf_cef_metabolome <- subset(cef_metabolome, infection == '630')[, cef_rf$feature]
inf_cef_metabolome$infection <- NULL
mock_cef_metabolome <- subset(cef_metabolome, infection == 'mock')[, cef_rf$feature]
mock_cef_metabolome$infection <- NULL
rm(cef_metabolome, cef_rf)
colnames(clinda_metabolome) <- make.names(colnames(clinda_metabolome))
inf_clinda_metabolome <- subset(clinda_metabolome, infection == '630')[, clinda_rf$feature]
inf_clinda_metabolome$infection<- NULL
mock_clinda_metabolome <- subset(clinda_metabolome, infection == 'mock')[, clinda_rf$feature]
mock_clinda_metabolome$infection<- NULL
rm(clinda_metabolome, clinda_rf)
colnames(strep_metabolome) <- make.names(colnames(strep_metabolome))
inf_strep_metabolome <- subset(strep_metabolome, infection == '630')[, strep_rf$feature]
inf_strep_metabolome$infection<- NULL
mock_strep_metabolome <- subset(strep_metabolome, infection == 'mock')[, strep_rf$feature]
mock_strep_metabolome$infection<- NULL
rm(strep_metabolome, strep_rf)

# Find significant differences
cef_inf_metab_pvalues <- c()
for (i in 1:ncol(inf_cef_metabolome)){cef_inf_metab_pvalues[i] <- wilcox.test(inf_cef_metabolome[,i], mock_cef_metabolome[,i], exact=FALSE)$p.value}
cef_inf_metab_pvalues <- as.character(round(p.adjust(cef_inf_metab_pvalues, method='BH'),3))
cef_inf_metab_names <- colnames(inf_cef_metabolome)
rm(inf_cef_metabolome)
clinda_inf_metab_pvalues <- c()
for (i in 1:ncol(inf_clinda_metabolome)){clinda_inf_metab_pvalues[i] <- wilcox.test(inf_clinda_metabolome[,i], mock_clinda_metabolome[,i], exact=FALSE)$p.value}
clinda_inf_metab_pvalues <- as.character(round(p.adjust(clinda_inf_metab_pvalues, method='BH'),3))
clinda_inf_metab_names <- colnames(inf_clinda_metabolome)
rm(inf_clinda_metabolome)
strep_inf_metab_pvalues <- c()
for (i in 1:ncol(inf_strep_metabolome)){strep_inf_metab_pvalues[i] <- wilcox.test(inf_strep_metabolome[,i], mock_strep_metabolome[,i], exact=FALSE)$p.value}
strep_inf_metab_pvalues <- as.character(round(p.adjust(strep_inf_metab_pvalues, method='BH'),3))
strep_inf_metab_names <- colnames(inf_strep_metabolome)
rm(inf_strep_metabolome)

```

***C. difficile* colonization induced distinct metabolomic changes of each pretreatment group.**
Following the changes to the cecal metabolome in response to antibiotic pretreatment, we assessed the degree to which *C. difficile* colonization altered the cecal metabolome of susceptible animals. We hypothesized that the introduction of a new competitor, *C. difficile*, would impact the metabolome either through signatures of its own metabolism or by causing a shift in the metabolism of other members in the surrounding community. First, in a similar approach to identifying susceptibility markers, we observed the global Bray-Curtis dissimilarities of mock-infected and *C. difficile*-infected metabolomes within each antibiotic pretreatment group individually. We compared the cecal metabolomes from all mice across pretreatment groups, and were unable to detect a consistent difference between groups (*p* = `r inf_p`; Fig. S3A). We moved on to individual antibiotic groups and found that both streptomycin (*p* = `r strep_inf_p`) and cefoperazone (*p* = `r cef_inf_p`) pretreated metabolomes deviated significantly from that of mock infection (Fig. 4A-4B). However, clindamycin-pretreated cecal metabolomes were not significantly altered by the presence of the pathogen (*p* = `r clinda_inf_p`; Fig. 4C). These results diverged from what was seen in the paired OTU relative abundance results where instead the cecal community structure of mice pretreated with cefoperazone was unchanged and clindamycin-pretreated communities were significantly different (Fig. S1). Interestingly, streptomycin-pretreated microbiomes were significantly altered by *C. difficile* infection at both the OTU and metabolomic levels (Fig. 4A & S1D). These data indicated that large shifts of populations in the cecal microbiota were not implicitly associated with concordant shifts in the metabolome. This supported the hypothesis that divergent community structures can ultimately share a convergent metabolic output despite changes to community structure and membership.





We then sought to identify changes in those metabolites that were potential markers of *C. difficile* infection through a unified analysis across all models tested. To accomplish this, we again applied the Random Forest machine-learning approach. We were able to distinguish those microbiomes infected with *C. difficile* from those that were not and identified the 10 metabolites with the greatest MDA (Fig. S3B). Only 5-aminovalerate appeared in more than one list as an optimal discriminatory metabolite so we compared its change in relative concentration in all pretreatments (Fig. 4G). This molecule is a known byproduct of D-proline Stickland fermentation by *C. difficile* [@Bouillaut2013;@Neumann-Schaal2015]. Presence of 5-aminovalerate appears to be dependent on bacterial processes as its relative concentrations were significantly lower in ceca of antibiotic-pretreated, mock-infected animals. Conversely, it was found to be highly abundant in most infected animals compared to both susceptible and resistant controls. Only in clindamycin-pretreatment, where CDI was quickly cleared, did the concentration of 5-aminovalerate remain near undetectable in a subset of samples. 

*C. difficile* is able to catabolize proline along with glycine through a set of paired biochemical reactions known as Stickland fermentation. 
Along these lines, when we measured for unified metabolite signals across all antibiotic pretreatment models, proline and 3 proline-conjugated amino acids (pro-hydroxy-pro, thioproline, and trans-4-hydroxyproline) were significantly decreased during infection (Fig. S3B; all *p* < `r infection_pvalues`). These combined results demonstrated that while distinct metabolic challenges may exist in each susceptible metabolome, Stickland fermentation could be a preferred energy acquisition pathway for *C. difficile* *in vivo*. Random Forest machine-learning models were able to reliably classify infected and uninfected mice in both streptomycin and cefoperazone pretreatments (Fig. 4A-4B, Out-of-bag error < 1%). However, infection did not provide for the same level of discrimination in clindamycin-pretreated mice (Fig. 4F, Out-of-bag error = 44.44%). None of the top 5 MDA metabolites in each antibiotic pretreatment were shared across all groups during individual analysis. These data supported our hypothesis that not only does each antibiotic pretreatment create different community structures, but that the metabolism of these distinct communities responds to *C. difficile* colonization in a unique manner. However, it was not possible to isolate where changes to the metabolome were a result of altered community metabolism, altered host metabolism, or from *C. difficile*.







**Infection corresponded with large shifts in the metatranscriptome strongly differ across antibiotic pretreatments.**
Combined, 16S rRNA gene sequencing and metabolomics results demonstrated that antibiotic pretreatments resulted in distinct bacterial communities that led to altered metabolite profiles. It is also possible that metabolism of the host or *C. difficile* contributed to the differences in the metabolomes during infection. To gain a more specific understanding of how the microbiota or *C. difficile* shaped the metabolic environment, we employed a metagenome-enabled metatranscriptomic shotgun sequencing approach with paired DNA and RNA samples collected from the cecal content of the mice used in the previous analyses. To achieve usable concentrations of bacterial mRNA, and also to obtain informative depths of sequencing  that adequately sampled the diversity in metagenomic and metatranscriptomic libraries (Fig. S6E&F), we were required to performed a combined sequencing effort for each treatment and infection group. To establish confidence in continuing pooled analysis, we calculated within-group sample variance replicates from cfu, otu relative abundance, and untargeted metabolomic (Table S5). Despite limited outliers, this revealed low levels in the median and interquartile range of variance within control and experimental groups at multiple levels of biology.

Following sequencing, metagenomic reads from mock-infected cecal communities were assembled *de novo* into contigs and putative genes were identified resulting in 234,868 (streptomycin), 83,534 (cefoperazone), and 35,681 (clindamycin) likely open reading frames in each metagenome. Of these putative genes, 28.5% were able to be annotated with known function KEGG, and many of these annotations were homologs among species within the dataset. Streptomycin pretreatment resulted in a significantly more diverse community (Fig. S1A) than other groups, so a more diverse metagenome was expected. 2408 unique functionally annotated genes were detected in the metagenome resulting from strptomycin pretreatment, at least 1163 more genes than either cefoperazone or clindamycin (Fig. S5). Also, as expected genes with any detectable transcript in any metatranscriptome were a subset of their corresponding metagenome. Metatranscriptomic read abundances were normalized to corresponding metagenomic coverage per gene followed by even subsampling of reads between conditions in order to create an equal standard of comparison across groups and identify genes with the largest increases in transcription above baseline. To highlight the largest magnitude of difference in transcription, we identified the top 15% genes (Fig. 5A) or pathways (Fig. 5B) in each antibiotic pretreatment group with the highest disparity in normalized cDNA abundance between mock and *C. difficile*-infected groups and cross-referenced the pretreatments. Based on the metabolomic results we hypothesized that pathways with the greatest differences would be related to catabolism of metabolites in established *C. difficile* nutrient niches.

```{r figure_5, echo=F, warning=F, message=F, results='hide', cache=F}

# Read in data
# Normalized Metatranscriptomes
cef_normalized_reads <- read.delim('data/read_mapping/cef_normalized_metaT.tsv', sep='\t', header=TRUE, stringsAsFactors=FALSE, row.names=7)
clinda_normalized_reads <- read.delim('data/read_mapping/clinda_normalized_metaT.tsv', sep='\t', header=TRUE, stringsAsFactors=FALSE, row.names=7)
strep_normalized_reads <- read.delim('data/read_mapping/strep_normalized_metaT.tsv', sep='\t', header=TRUE, stringsAsFactors=FALSE, row.names=7)

# Format data
# Remove C. difficile genes
cdiff_omit <- c('cdf','pdc','cdc','cdl','pdf')
cef_normalized_reads <- cef_normalized_reads[!cef_normalized_reads$organism %in% cdiff_omit,]
clinda_normalized_reads <- clinda_normalized_reads[!clinda_normalized_reads$organism %in% cdiff_omit,]
strep_normalized_reads <- strep_normalized_reads[!strep_normalized_reads$organism %in% cdiff_omit,]
rm(cdiff_omit)
# Screen for those genes that have a gene annotation
cef_annotated <- cef_normalized_reads[!rownames(cef_normalized_reads) %in% rownames(cef_normalized_reads[grep('unknown_\\d', rownames(cef_normalized_reads)),]), ]
clinda_annotated <- clinda_normalized_reads[!rownames(clinda_normalized_reads) %in% rownames(clinda_normalized_reads[grep('unknown_\\d', rownames(clinda_normalized_reads)),]), ]
strep_annotated <- strep_normalized_reads[!rownames(strep_normalized_reads) %in% rownames(strep_normalized_reads[grep('unknown_\\d', rownames(strep_normalized_reads)),]), ]
rm(cef_normalized_reads, clinda_normalized_reads, strep_normalized_reads)
# Screen for genes with no reads
cef_annotated[,2:3] <- round(cef_annotated[,2:3])
cef_annotated <- subset(cef_annotated, rowSums(cef_annotated[,2:3]) != 0)
clinda_annotated[,2:3] <- round(clinda_annotated[,2:3])
clinda_annotated <- subset(clinda_annotated, rowSums(clinda_annotated[,2:3]) != 0)
strep_annotated[,2:3] <- round(strep_annotated[,2:3])
strep_annotated <- subset(strep_annotated, rowSums(strep_annotated[,2:3]) != 0)
# Screen out ribosomal genes + hypothetical, putative, and uncharacterized annotations
cef_annotated <- subset(cef_annotated, !grepl('Ribosomal_RNA*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('ribosomal_RNA*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('*ribosomal_RNA*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('*ribosomal_protein*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('16S_rRNA_*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('23S_rRNA_*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('uncharacterized_*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('putative_*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, !grepl('Predicted_*', cef_annotated$description))
cef_annotated <- subset(cef_annotated, description != 'hypothetical_protein')
clinda_annotated <- subset(clinda_annotated, !grepl('Ribosomal_RNA*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('ribosomal_RNA*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('*ribosomal_RNA*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('*ribosomal_protein*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('16S_rRNA_*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('23S_rRNA_*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('uncharacterized_*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('putative_*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, !grepl('Predicted_*', clinda_annotated$description))
clinda_annotated <- subset(clinda_annotated, description != 'hypothetical_protein')
strep_annotated <- subset(strep_annotated, !grepl('Ribosomal_RNA*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('ribosomal_RNA*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('*ribosomal_RNA*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('*ribosomal_protein*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('16S_rRNA_*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('23S_rRNA_*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('uncharacterized_*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('putative_*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, !grepl('Predicted_*', strep_annotated$description))
strep_annotated <- subset(strep_annotated, description != 'hypothetical_protein')
# Simplify column names
colnames(cef_annotated) <- c('gene', 'cef_630_reads', 'cef_mock_reads','organism','description','ko','pathways')
colnames(clinda_annotated) <- c('gene', 'clinda_630_reads', 'clinda_mock_reads','organism','description','ko','pathways')
colnames(strep_annotated) <- c('gene', 'strep_630_reads', 'strep_mock_reads','organism','description','ko','pathways')

# Calculate and take top differences
cef_10 <- round(nrow(cef_annotated) * 0.15)
clinda_10 <- round(nrow(clinda_annotated) * 0.15)
strep_10 <- round(nrow(strep_annotated) * 0.15)
cef_annotated$diff <- abs(cef_annotated$cef_630_reads - cef_annotated$cef_mock_reads)
cef_annotated <- cef_annotated[order(-cef_annotated$diff),]
top_cef_genes <- cef_annotated$gene[1:cef_10]
cef_annotated$diff <- NULL
clinda_annotated$diff <- abs(clinda_annotated$clinda_630_reads - clinda_annotated$clinda_mock_reads)
clinda_annotated <- clinda_annotated[order(-clinda_annotated$diff),]
top_clinda_genes <- clinda_annotated$gene[1:clinda_10]
clinda_annotated$diff <- NULL
strep_annotated$diff <- abs(strep_annotated$strep_630_reads - strep_annotated$strep_mock_reads)
strep_annotated <- strep_annotated[order(-strep_annotated$diff),]
top_strep_genes <- strep_annotated$gene[1:strep_10]
strep_annotated$diff <- NULL
rm(cef_10, clinda_10, strep_10)

# Calculate overlap of top differences
top_cef_clinda_genes <- intersect(top_cef_genes, top_clinda_genes)
top_cef_strep_genes <- intersect(top_cef_genes, top_strep_genes)
top_clinda_strep_genes <- intersect(top_clinda_genes, top_strep_genes)
top_genes <- intersect(top_cef_genes, intersect(top_clinda_genes, top_strep_genes))
top_cef_genes <- subset(top_cef_genes, !top_cef_genes %in% top_cef_clinda_genes)
top_cef_genes <- subset(top_cef_genes, !top_cef_genes %in% top_cef_strep_genes)
top_cef_genes <- subset(top_cef_genes, !top_cef_genes %in% top_genes)
top_clinda_genes <- subset(top_clinda_genes, !top_clinda_genes %in% top_cef_clinda_genes)
top_clinda_genes <- subset(top_clinda_genes, !top_clinda_genes %in% top_clinda_strep_genes)
top_clinda_genes <- subset(top_clinda_genes, !top_clinda_genes %in% top_genes)
top_strep_genes <- subset(top_strep_genes, !top_strep_genes %in% top_clinda_strep_genes)
top_strep_genes <- subset(top_strep_genes, !top_strep_genes %in% top_cef_strep_genes)
top_strep_genes <- subset(top_strep_genes, !top_strep_genes %in% top_genes)
top_cef_clinda_genes <- as.character(as.numeric(length(top_cef_clinda_genes)))
top_cef_strep_genes <- as.character(as.numeric(length(top_cef_strep_genes)))
top_clinda_strep_genes <- as.character(as.numeric(length(top_clinda_strep_genes)))
top_all_genes <- as.character(as.numeric(length(top_genes)))
top_cef_genes <- as.character(as.numeric(length(top_cef_genes)))
top_clinda_genes <- as.character(as.numeric(length(top_clinda_genes)))
top_strep_genes <- as.character(as.numeric(length(top_strep_genes)))
rm(top_genes)

# Aggregate pathways
cef_pathways <- subset(cef_annotated, !is.na(cef_annotated$pathways))
cef_pathways <- aggregate(cbind(cef_pathways$cef_630_reads, cef_pathways$cef_mock_reads), by=list(cef_pathways$pathways), FUN=sum)
rownames(cef_pathways) <- cef_pathways$Group.1
cef_pathways$Group.1 <- NULL
cef_pathways <- subset(cef_pathways, !rownames(cef_pathways) %in% c('Ribosome:Translation','Metabolic_pathways'))
colnames(cef_pathways) <- c('cef_630_reads', 'cef_mock_reads')
clinda_pathways <- subset(clinda_annotated, !is.na(clinda_annotated$pathways))
clinda_pathways <- aggregate(cbind(clinda_pathways$clinda_630_reads, clinda_pathways$clinda_mock_reads), by=list(clinda_pathways$pathways), FUN=sum)
rownames(clinda_pathways) <- clinda_pathways$Group.1
clinda_pathways$Group.1 <- NULL
clinda_pathways <- subset(clinda_pathways, !rownames(clinda_pathways) %in% c('Ribosome:Translation','Metabolic_pathways'))
colnames(clinda_pathways) <- c('clinda_630_reads', 'clinda_mock_reads')
strep_pathways <- subset(strep_annotated, !is.na(strep_annotated$pathways))
strep_pathways <- aggregate(cbind(strep_pathways$strep_630_reads, strep_pathways$strep_mock_reads), by=list(strep_pathways$pathways), FUN=sum)
rownames(strep_pathways) <- strep_pathways$Group.1
strep_pathways$Group.1 <- NULL
strep_pathways <- subset(strep_pathways, !rownames(strep_pathways) %in% c('Ribosome:Translation','Metabolic_pathways'))
colnames(strep_pathways) <- c('strep_630_reads', 'strep_mock_reads')
rm(cef_annotated, clinda_annotated, strep_annotated)

# Calculate and take top differences
cef_10 <- round(nrow(cef_pathways) * 0.15)
clinda_10 <- round(nrow(clinda_pathways) * 0.15)
strep_10 <- round(nrow(strep_pathways) * 0.15)
cef_pathways$diff <- abs(cef_pathways$cef_630_reads - cef_pathways$cef_mock_reads)
cef_pathways <- cef_pathways[order(-cef_pathways$diff),]
top_cef_paths <- rownames(cef_pathways[1:cef_10,])
cef_pathways$diff <- NULL
clinda_pathways$diff <- abs(clinda_pathways$clinda_630_reads - clinda_pathways$clinda_mock_reads)
clinda_pathways <- clinda_pathways[order(-clinda_pathways$diff),]
top_clinda_paths <- rownames(clinda_pathways[1:clinda_10,])
clinda_pathways$diff <- NULL
strep_pathways$diff <- abs(strep_pathways$strep_630_reads - strep_pathways$strep_mock_reads)
strep_pathways <- strep_pathways[order(-strep_pathways$diff),]
top_strep_paths <- rownames(strep_pathways[1:strep_10,])
strep_pathways$diff <- NULL
rm(cef_10, clinda_10, strep_10,
   cef_pathways, clinda_pathways, strep_pathways)

# Calculate overlap of top differences
top_cef_clinda_paths <- intersect(top_cef_paths, top_clinda_paths)
top_cef_strep_paths <- intersect(top_cef_paths, top_strep_paths)
top_clinda_strep_paths <- intersect(top_clinda_paths, top_strep_paths)
top_paths <- intersect(top_cef_paths, intersect(top_clinda_paths, top_strep_paths))
top_cef_paths <- subset(top_cef_paths, !top_cef_paths %in% top_cef_clinda_paths)
top_cef_paths <- subset(top_cef_paths, !top_cef_paths %in% top_cef_strep_paths)
top_cef_paths <- subset(top_cef_paths, !top_cef_paths %in% top_paths)
top_clinda_paths <- subset(top_clinda_paths, !top_clinda_paths %in% top_cef_clinda_paths)
top_clinda_paths <- subset(top_clinda_paths, !top_clinda_paths %in% top_clinda_strep_paths)
top_clinda_paths <- subset(top_clinda_paths, !top_clinda_paths %in% top_paths)
top_strep_paths <- subset(top_strep_paths, !top_strep_paths %in% top_clinda_strep_paths)
top_strep_paths <- subset(top_strep_paths, !top_strep_paths %in% top_cef_strep_paths)
top_strep_paths <- subset(top_strep_paths, !top_strep_paths %in% top_paths)
top_cef_clinda_paths <- as.character(as.numeric(length(top_cef_clinda_paths)))
top_cef_strep_paths <- as.character(as.numeric(length(top_cef_strep_paths)))
top_clinda_strep_paths <- as.character(as.numeric(length(top_clinda_strep_paths)))
top_all_paths <- as.character(as.numeric(length(top_paths)))
top_cef_paths <- as.character(as.numeric(length(top_cef_paths)))
top_clinda_paths <- as.character(as.numeric(length(top_clinda_paths)))
top_strep_paths <- as.character(as.numeric(length(top_strep_paths)))
rm(top_paths)

```

This analysis revealed modest conservation of transcriptional changes induced by *C. difficile* colonization across the antibiotic pretreatments. This trend was partially driven by significant differences in community membership (Fig. 2A), and consequently the metagenomic repertoire available for transcription. The most stark finding was that only 8 genes were among those shared across pretreatment groups (Fig. 5A). Although mice receiving the streptomycin and clindamycin pretreatments shared a large number of altered genes, cefoperazone-pretreated mice shared relatively few with the metatranscriptome of the other pretreament groups. This result was likely due to large differences in community compositions. Upon inspection of the `r top_all_genes` genes that were shared across the three pretreatment groups, a Dps family protein gene was the most varied in transcription. The protein product of this gene is associated with DNA protection and iron binding under periods of environmental stress [@Ilari2000]. Furthermore, formate acetyltransferase and phosphopyruvate hydratase are highly varied in their transcription and are central nodes of metabolism, which lend themselves to the hypothesis that the microbiota must process energy sources differently in the presence of *C. difficile*. Pathway-level analysis revealed a larger proportion of overlap between pretreatments than at the gene-level (Fig. 5B). Among these differences, the greatest degree of change was in ABC transporter transcription further supported large-scale shifts in nutrient import across the individual communities. More to this point, numerous genes for transport of simple carbohydrates and glycolysis were also differentially overrepresented in infected conditions across pretreatment groups when compared to mock-infection mice (Fig. S4). Among other pathways in the shared differences grouping, the microbial metabolism in diverse environments KEGG category is a broad classification that includes genes for not only carbohydrate and amino acid catabolism, but also the degradation and metabolism of xenobiotics. Inclusion of this category supports the hypothesis that the microbiota may be restructuring energy acquisition strategies or dealing with additional metabolites introduced due to pathogen colonization. Parallel to these results, large changes in oxidative phosphorylation as well as starch/sucrose metabolism also indicate modification of nutrient niche segregation between mock and infected states that may drive the ability of *C. difficile* to persist. Together, these shifts demonstrated that *C. difficile* colonization is associated with substantial changes in the transcription of nutrient acquisition pathways, which vary greatly between antibiotic-induced susceptible states.

```{r figure_6, echo=F, warning=F, message=F, results='hide', cache=F}

# Read in data
# Normalized Metatranscriptomes
cef_normalized_reads <- read.delim('data/read_mapping/cef_normalized_metaT.tsv', sep='\t', header=TRUE, row.names=7)
clinda_normalized_reads <- read.delim('data/read_mapping/clinda_normalized_metaT.tsv', sep='\t', header=TRUE, row.names=7)
strep_normalized_reads <- read.delim('data/read_mapping/strep_normalized_metaT.tsv', sep='\t', header=TRUE, row.names=7)
# KEGG organism file
kegg_tax <- read.delim('data/kegg/kegg_taxonomy.tsv', sep='\t', header=TRUE)
kegg_tax[] <- lapply(kegg_tax, as.character)
# 16S data
genus_shared <- read.delim('data/16S_analysis/all_treatments.0.03.unique_list.0.03.filter.0.03.subsample.shared', sep='\t', header=TRUE, row.names=2)
genus_shared$label <- NULL
genus_shared$numOtus <- NULL
genus_tax <- read.delim('data/16S_analysis/all_treatments.genus.final.taxonomy', sep='\t', header=TRUE)
rownames(genus_tax) <- genus_tax$OTU
genus_tax$OTU <- NULL
# Metadata
metadata <- read.delim('data/metadata.tsv', sep='\t', header=TRUE, row.names=1)

# Format transcription
cef_normalized_reads$gene <- NULL
cef_normalized_reads[,c(1:2)] <- log2(cef_normalized_reads[,c(1:2)] + 1)
clinda_normalized_reads$gene <- NULL
clinda_normalized_reads[,c(1:2)] <- log2(clinda_normalized_reads[,c(1:2)] + 1)
strep_normalized_reads$gene <- NULL
strep_normalized_reads[,c(1:2)] <- log2(strep_normalized_reads[,c(1:2)] + 1)
# Remove those genes without an organism annotation
cef_annotated <- cef_normalized_reads[!is.na(cef_normalized_reads$organism),]
clinda_annotated <- clinda_normalized_reads[!is.na(clinda_normalized_reads$organism),]
strep_annotated <- strep_normalized_reads[!is.na(strep_normalized_reads$organism),]
rm(cef_normalized_reads, clinda_normalized_reads, strep_normalized_reads)
# Remove any C. difficile genes with transcription only in infected
cdiff_omit <- c('cdf','pdc','cdc','cdl','pdf')
cef_annotated <- subset(cef_annotated, !(cef_annotated$organism %in% cdiff_omit))
clinda_annotated <- subset(clinda_annotated, !(clinda_annotated$organism %in% cdiff_omit))
strep_annotated <- subset(strep_annotated, !(strep_annotated$organism %in% cdiff_omit))
rm(cdiff_omit)
# Remove all possible mammalian genes
mamm_omit <- c('fab','cfa','ggo','hgl','hsa','mcc','mdo','pon','aml',
               'ptr','rno','shr','ssc','aml','bta','cge','ecb',
               'pps','fca','mmu','oaa','gga','ola','acs','aga')
strep_annotated <- subset(strep_annotated, !(strep_annotated$organism %in% mamm_omit))
cef_annotated <- subset(cef_annotated, !(cef_annotated$organism %in% mamm_omit))
clinda_annotated <- subset(clinda_annotated, !(clinda_annotated$organism %in% mamm_omit))
rm(mamm_omit)
# Calculate correlation coefficients
strep_corr <- as.character(round(cor.test(strep_annotated[,2], strep_annotated[,1], method='spearman', exact=FALSE)$estimate, digits=3))
cef_corr <- as.character(round(cor.test(cef_annotated[,2], cef_annotated[,1], method='spearman', exact=FALSE)$estimate, digits=3))
clinda_corr <- as.character(round(cor.test(clinda_annotated[,2], clinda_annotated[,1], method='spearman', exact=FALSE)$estimate, digits=3))

# Collate transcript abundances at genus-level 
genus_shared <- t(genus_shared)
genus_tax_shared <- merge(genus_tax, genus_shared, by='row.names')
genus_tax_shared$Row.names <- NULL
genus_tax_agg_shared <- aggregate(. ~ Genus, data=genus_tax_shared, FUN=sum)
rownames(genus_tax_agg_shared) <- genus_tax_agg_shared$Genus
genus_tax_agg_shared$Genus <- NULL
genus_tax_agg_shared <- t(genus_tax_agg_shared)
rm(genus_tax, genus_shared, genus_tax_shared)

# Combine with metadata and collate within treatment
genus_tax_agg_metadata_shared <- merge(metadata, genus_tax_agg_shared, by='row.names')
genus_tax_agg_metadata_shared$Row.names <- NULL
genus_tax_agg_metadata_shared$cage <- NULL
genus_tax_agg_metadata_shared$mouse <- NULL
genus_tax_agg_metadata_shared$gender <- NULL
genus_tax_agg_metadata_shared$type <- NULL
genus_tax_agg_metadata_shared$susceptibility <- NULL
genus_tax_agg_metadata_shared$infection <- NULL
genus_tax_agg_metadata_shared <- subset(genus_tax_agg_metadata_shared, genus_tax_agg_metadata_shared$abx %in% c('cefoperazone','clindamycin','streptomycin'))
genus_tax_agg_metadata_shared <- aggregate(. ~ abx, data=genus_tax_agg_metadata_shared, FUN=median)
rownames(genus_tax_agg_metadata_shared) <- genus_tax_agg_metadata_shared$abx
genus_tax_agg_metadata_shared$abx <- NULL
genus_tax_agg_metadata_shared <- (genus_tax_agg_metadata_shared / rowSums(genus_tax_agg_metadata_shared)) * 100
genus_tax_agg_metadata_shared <- as.data.frame(t(genus_tax_agg_metadata_shared))
cef_genus <- genus_tax_agg_metadata_shared
cef_genus$clindamycin <- NULL
cef_genus$streptomycin <- NULL
clinda_genus <- genus_tax_agg_metadata_shared
clinda_genus$cefoperazone <- NULL
clinda_genus$streptomycin <- NULL
strep_genus <- genus_tax_agg_metadata_shared
strep_genus$clindamycin <- NULL
strep_genus$cefoperazone <- NULL
rm(genus_tax_agg_shared, genus_tax_agg_metadata_shared, metadata)

# Using previously defined lines, find outliers to y = x
strep_630_outliers <- subset(strep_annotated, strep_annotated$strep_630_metaT_reads > strep_annotated$strep_mock_metaT_reads + 2)
strep_mock_outliers <- subset(strep_annotated, strep_annotated$strep_mock_metaT_reads > strep_annotated$strep_630_metaT_reads + 2)
cef_630_outliers <- subset(cef_annotated, cef_annotated$cef_630_metaT_reads > cef_annotated$cef_mock_metaT_reads + 2)
cef_mock_outliers <- subset(cef_annotated, cef_annotated$cef_mock_metaT_reads > cef_annotated$cef_630_metaT_reads + 2)
clinda_630_outliers <- clinda_annotated[(clinda_annotated$clinda_630_metaT_reads > clinda_annotated$clinda_mock_metaT_reads + 2),]
clinda_mock_outliers <- clinda_annotated[(clinda_annotated$clinda_mock_metaT_reads > clinda_annotated$clinda_630_metaT_reads + 2),]
rm(strep_annotated, cef_annotated, clinda_annotated)

# Calculate mean distance of outliers to x=y
dists_630 <- c()
for (i in 1:nrow(strep_630_outliers)){
  dists_630[i] <- dist_xy(c(strep_630_outliers$strep_630_metaT_reads[i], strep_630_outliers$strep_mock_metaT_reads[i]))
}
dists_mock <- c()
for (i in 1:nrow(strep_mock_outliers)){
  dists_mock[i] <- dist_xy(c(strep_mock_outliers$strep_630_metaT_reads[i], strep_mock_outliers$strep_mock_metaT_reads[i]))
}
strep_mean_dist <- as.character(round(mean(c(dists_630, dists_mock)), 3))
strep_outs <- as.character(as.numeric(length(c(dists_630, dists_mock))))
dists_630 <- c()
for (i in 1:nrow(cef_630_outliers)){
  dists_630[i] <- dist_xy(c(cef_630_outliers$cef_630_metaT_reads[i], cef_630_outliers$cef_mock_metaT_reads[i]))
}
dists_mock <- c()
for (i in 1:nrow(cef_mock_outliers)){
  dists_mock[i] <- dist_xy(c(cef_mock_outliers$cef_630_metaT_reads[i], cef_mock_outliers$cef_mock_metaT_reads[i]))
}
cef_mean_dist <- as.character(round(mean(c(dists_630, dists_mock)), 3))
cef_outs <- as.character(as.numeric(length(c(dists_630, dists_mock))))
dists_630 <- c()
for (i in 1:nrow(clinda_630_outliers)){
  dists_630[i] <- dist_xy(c(clinda_630_outliers$clinda_630_metaT_reads[i], clinda_630_outliers$clinda_mock_metaT_reads[i]))
}
dists_mock <- c()
for (i in 1:nrow(clinda_mock_outliers)){
  dists_mock[i] <- dist_xy(c(clinda_mock_outliers$clinda_630_metaT_reads[i], clinda_mock_outliers$clinda_mock_metaT_reads[i]))
}
clinda_mean_dist <- as.character(round(mean(c(dists_630, dists_mock)), 3))
clinda_outs <- as.character(as.numeric(length(c(dists_630, dists_mock))))
rm(dists_630, dists_mock, i)

# Drop levels
strep_630_outliers[] <- lapply(strep_630_outliers, as.character)
strep_mock_outliers[] <- lapply(strep_mock_outliers, as.character)
cef_630_outliers[] <- lapply(cef_630_outliers, as.character)
cef_mock_outliers[] <- lapply(cef_mock_outliers, as.character)
clinda_630_outliers[] <- lapply(clinda_630_outliers, as.character)
clinda_mock_outliers[] <- lapply(clinda_mock_outliers, as.character)

# Save KEGG ID names
strep_630_outliers$kegg_id <- rownames(strep_630_outliers)
strep_mock_outliers$kegg_id <- rownames(strep_mock_outliers)
cef_630_outliers$kegg_id <- rownames(cef_630_outliers)
cef_mock_outliers$kegg_id <- rownames(cef_mock_outliers)
clinda_630_outliers$kegg_id <- rownames(clinda_630_outliers)
clinda_mock_outliers$kegg_id <- rownames(clinda_mock_outliers)

# Merge with KEGG taxonomy
strep_630_outliers <- merge(x=strep_630_outliers, y=kegg_tax, by.x='organism', by.y='org_code', all.x=TRUE)
strep_630_outliers$org_code <- NULL
strep_630_outliers$strep_630_metaT_reads <- as.numeric(strep_630_outliers$strep_630_metaT_reads)
strep_630_outliers$strep_mock_metaT_reads <- as.numeric(strep_630_outliers$strep_mock_metaT_reads)
strep_mock_outliers <- merge(x=strep_mock_outliers, y=kegg_tax, by.x='organism', by.y='org_code', all.x=TRUE)
strep_mock_outliers$org_code <- NULL
strep_mock_outliers$strep_630_metaT_reads <- as.numeric(strep_mock_outliers$strep_630_metaT_reads)
strep_mock_outliers$strep_mock_metaT_reads <- as.numeric(strep_mock_outliers$strep_mock_metaT_reads)
cef_630_outliers <- merge(x=cef_630_outliers, y=kegg_tax, by.x='organism', by.y='org_code', all.x=TRUE)
cef_630_outliers$org_code <- NULL
cef_630_outliers$cef_630_metaT_reads <- as.numeric(cef_630_outliers$cef_630_metaT_reads)
cef_630_outliers$cef_mock_metaT_reads <- as.numeric(cef_630_outliers$cef_mock_metaT_reads)
cef_mock_outliers <- merge(x=cef_mock_outliers, y=kegg_tax, by.x='organism', by.y='org_code', all.x=TRUE)
cef_mock_outliers$org_code <- NULL
cef_mock_outliers$cef_630_metaT_reads <- as.numeric(cef_mock_outliers$cef_630_metaT_reads)
cef_mock_outliers$cef_mock_metaT_reads <- as.numeric(cef_mock_outliers$cef_mock_metaT_reads)
clinda_630_outliers <- merge(x=clinda_630_outliers, y=kegg_tax, by.x='organism', by.y='org_code', all.x=TRUE)
clinda_630_outliers$org_code <- NULL
clinda_630_outliers$clinda_630_metaT_reads <- as.numeric(clinda_630_outliers$clinda_630_metaT_reads)
clinda_630_outliers$clinda_mock_metaT_reads <- as.numeric(clinda_630_outliers$clinda_mock_metaT_reads)
clinda_mock_outliers <- merge(x=clinda_mock_outliers, y=kegg_tax, by.x='organism', by.y='org_code', all.x=TRUE)
clinda_mock_outliers$org_code <- NULL
clinda_mock_outliers$clinda_630_metaT_reads <- as.numeric(clinda_mock_outliers$clinda_630_metaT_reads)
clinda_mock_outliers$clinda_mock_metaT_reads <- as.numeric(clinda_mock_outliers$clinda_mock_metaT_reads)
rm(kegg_tax)

# Find difference in expression for outliers
strep_outliers <- rbind(strep_630_outliers, strep_mock_outliers)
strep_difference <- abs(((2^strep_outliers$strep_mock_metaT_reads) - 1) - ((2^strep_outliers$strep_630_metaT_reads) - 1))
strep_difference <- as.data.frame(cbind(strep_outliers$genus, strep_difference))
colnames(strep_difference) <- c('genus', 'difference')
strep_gene_count <- as.data.frame(table(strep_difference$genus))
colnames(strep_gene_count) <- c('genus', 'count')
strep_difference <- aggregate(. ~ genus, data=strep_difference, FUN=sum)
strep_difference <- merge(strep_difference, strep_gene_count, by='genus')
strep_difference$difference <- strep_difference$difference / strep_difference$count
strep_difference$count <- NULL
rm(strep_gene_count, strep_630_outliers, strep_mock_outliers)
strep_difference$difference <- log2(as.numeric(as.character(strep_difference$difference)) + 1)
strep_genus_diff <- merge(strep_genus, strep_difference, by.x='row.names', by.y='genus')
rownames(strep_genus_diff) <- strep_genus_diff$Row.names
strep_genus_diff$Row.names <- NULL
colnames(strep_genus_diff) <- c('relAbund','transcriptChange')
rm(strep_difference, strep_genus)
cef_outliers <- rbind(cef_630_outliers, cef_mock_outliers)
cef_difference <- abs(((2^cef_outliers$cef_mock_metaT_reads) - 1) - ((2^cef_outliers$cef_630_metaT_reads) - 1))
cef_difference <- as.data.frame(cbind(cef_outliers$genus, cef_difference))
colnames(cef_difference) <- c('genus', 'difference')
cef_gene_count <- as.data.frame(table(cef_difference$genus))
colnames(cef_gene_count) <- c('genus', 'count')
cef_difference <- aggregate(. ~ genus, data=cef_difference, FUN=sum)
cef_difference <- merge(cef_difference, cef_gene_count, by='genus')
cef_difference$difference <- cef_difference$difference / cef_difference$count
cef_difference$count <- NULL
rm(cef_gene_count, cef_630_outliers, cef_mock_outliers)
cef_difference$difference <- log2(as.numeric(as.character(cef_difference$difference)) + 1)
cef_genus_diff <- merge(cef_genus, cef_difference, by.x='row.names', by.y='genus')
rownames(cef_genus_diff) <- cef_genus_diff$Row.names
cef_genus_diff$Row.names <- NULL
colnames(cef_genus_diff) <- c('relAbund','transcriptChange')
rm(cef_difference, cef_genus)
clinda_outliers <- rbind(clinda_630_outliers, clinda_mock_outliers)
clinda_difference <- abs(((2^clinda_outliers$clinda_mock_metaT_reads) - 1) - ((2^clinda_outliers$clinda_630_metaT_reads) - 1))
clinda_difference <- as.data.frame(cbind(clinda_outliers$genus, clinda_difference))
colnames(clinda_difference) <- c('genus', 'difference')
clinda_gene_count <- as.data.frame(table(clinda_difference$genus))
colnames(clinda_gene_count) <- c('genus', 'count')
clinda_difference <- aggregate(. ~ genus, data=clinda_difference, FUN=sum)
clinda_difference <- merge(clinda_difference, clinda_gene_count, by='genus')
clinda_difference$difference <- clinda_difference$difference / clinda_difference$count
clinda_difference$count <- NULL
rm(clinda_gene_count, clinda_630_outliers, clinda_mock_outliers)
clinda_difference$difference <- log2(as.numeric(as.character(clinda_difference$difference)) + 1)
clinda_genus_diff <- merge(clinda_genus, clinda_difference, by.x='row.names', by.y='genus')
rownames(clinda_genus_diff) <- clinda_genus_diff$Row.names
clinda_genus_diff$Row.names <- NULL
colnames(clinda_genus_diff) <- c('relAbund','transcriptChange')
rm(clinda_difference, clinda_genus)
rm(strep_outliers, cef_outliers, clinda_outliers)

# Spearman correlation of 16S and metatransctiptomic change
strep_16S_r <- as.character(round(cor.test(strep_genus_diff$transcriptChange, strep_genus_diff$relAbund, method='spearman', exact=FALSE)$estimate, 3))
strep_16S_p <- as.character(round(cor.test(strep_genus_diff$transcriptChange, strep_genus_diff$relAbund, method='spearman', exact=FALSE)$p.value, 3))
cef_16S_r <- as.character(round(cor.test(cef_genus_diff$transcriptChange, cef_genus_diff$relAbund, method='spearman', exact=FALSE)$estimate, 3))
cef_16S_p <- as.character(round(cor.test(cef_genus_diff$transcriptChange, cef_genus_diff$relAbund, method='spearman', exact=FALSE)$p.value, 3))
clinda_16S_r <- as.character(round(cor.test(clinda_genus_diff$transcriptChange, clinda_genus_diff$relAbund, method='spearman', exact=FALSE)$estimate, 3))
clinda_16S_p <- as.character(round(cor.test(clinda_genus_diff$transcriptChange, clinda_genus_diff$relAbund, method='spearman', exact=FALSE)$p.value, 3))
rm(strep_genus_diff, cef_genus_diff, clinda_genus_diff)

```

***C. difficile* colonization disproportionately impacted the gene transcription of low-abundance bacterial taxa.**
As not all bacterial taxa share identical metabolic capabilities, we hypothesized that transcription in specific subsets of bacteria were differentially affected by the presence or metabolic activity of *C. difficile*. We sought to delineate the transcriptomic contributions of separate bacterial taxa within each metatranscriptomic dataset. To accomplish this we utilized the genus level taxonomic information associated with each KEGG annotation to identify which group likely contributed a given gene to the metagenome. Many genes in the KEGG database are annotated as hypothetical or uncharacterized but still possess a taxonomic annotation. This resulted in substantially more genes from the total being conserved for analysis in each group. With these data, we narrowed the focus to transcription for genera that represented >0.01% of genes receiving taxonomic annotations in any of the metagenomic assemblies. Final metatranscriptomic abundances were then obtained by dividing the cDNA totals of each gene by its metagenomic coverage in order to normalize for abundance of the contributing bacterial taxa. We then directly compared the normalized cDNA abundances for each gene between infected and uninfected states for each antibiotic pretreatment and applied Spearman correlation to identify distinct patterns of transcription (Fig. 6A-C). This resulted in `r strep_outs` genes that had an average distance of `r strep_mean_dist` units of deviation (UD) associated with streptomycin-pretreatment, `r cef_outs` genes at an average distance `r cef_mean_dist` UD with cefoperazone-pretreatment, and only `r clinda_outs` outliers at an average distance of `r clinda_mean_dist` UD in clindamycin-pretreatment. Overall, the clindamycin pretreatment was associated with the fewest transcription outliers between uninfected and infection conditions compared with those of the other antibiotic groups. This indicated that the gene transcription in the clindamycin-pretreated community was not significantly altered in the presence of *C. difficile*.

Since many genes lack a specific functional annotation in KEGG but do map to individual bacterial species, we continued the analysis by measuring change in transcript abundances contributed by the more prevalent genera. Incorporation of the genus-level taxonomic information for each transcript revealed that outlier genes were contributed by underrepresented genera. In streptomycin-pretreated mice, the most prominent differences were in 937 genes belonging to *Lactobacillus* with increased transcription during *C. difficile* infection (Fig. 6A). In cefoperazone-pretreated mice 2290 genes belonging to *Bacteroides* had higher transcription in mock infected mice (Fig. 6B). A consistent trend in streptomycin and cefoperazone pretreatments was an overrepresentation of highly transcribed genes from genera belonging to *Bacteroidetes* during mock infection. The metatransciptomes within both of these pretreatment conditions poorly correlated between mock and infected conditions, indicating a high degree of change induced by *C. difficile* colonization (*R* = `r strep_corr` & *R* = `r cef_corr`). In clindamycin-pretreated mice the largest difference in transcription was for 510 *Lactobacillus* genes with increased transcription during CDI (Fig. 6C). Infected and uninfected metatranscriptomes associated with this antibiotic more strongly correlated than either other pretreatment (*R* = `r clinda_corr`). This suggested that the degree to which the metatranscriptome was shifted by infection corresponded to prolonged colonization. We further characterized this by tabulating the absolute difference between mock-infection and CDI in all transcriptomic abundances per genus in each antibiotic pretreatment, normalizing for the number of genes detected in each genera. Taxa were then separated into groups based on their relative abundance in each community based on 16S rRNA gene sequencing (Fig. 6D). This revealed that most change occurred within relatively rare bacterial taxa. As the number and diversity of low abundance taxa was much larger than that of highly abundant groups, the majority of unique genes and metabolic potential was held within the minority of each community. Consequently, the metatranscriptomic differences in these groups may drive a disproportionately large change in the overall environment of the intestine as a function of their collective metabolism.


**Discussion**

Our results demonstrate that distinct intestinal ecosystems are differently impacted by *C. difficile* colonization and that these changes to community metabolism could have implications on the ability of the pathogen to persist in those environments. Furthermore, discordant modifications shortly after infection to OTU abundances, metatranscriptomic activity, and metabolome composition suggest that *C. difficile* manipulates the niche landscape of the intestinal tract in order to better maintain long-term colonization even prior to the onset of toxin-mediated inflammation. This hypothesis is best supported by the disparity of community-level phenotypes between the clindamycin and cefoperazone or streptomycin pretreatment models. Only in the clindamycin pretreatment model were animals able to go on to clear *C. difficile*. Unlike the other conditions, clindamycin-pretreated communities were significantly altered in community structure following *C. difficile* colonization, but not at the metabolomic level (Fig. S1F & 4A). This disconnect between community structure and metabolic output was explained by their similarity in gene transcription (Fig. 4D & 5C). Collectively, these results advance the idea that for *C. difficile* to maintain colonization it must exploit and restructure the available niche space. 

Instances of active nutrient niche restructuring in the gut have been documented previously for prominent symbiotic bacterial species in gnotobiotic mice [@Mahowald2009], but never before in a conventionally-reared animal model of infection following antibiotic pretreatment. Interestingly, the taxonomic groups that were represented as outliers in the normalized metatranscriptomes of each antibiotic pretreatments were rare in their cecal community (on average less than 5% of the community; Fig. 2A & 6). Previous studies have found that rare taxonomic groups, even those at a low abundance as a result of a spontaneous perturbation, may have disproportion effects on the metabolic environment of the community at large [@Jousset2017]. For example, this strategy has been observed in temperate lakes where conditionally rare microbes were found to be far more metabolically active than highly abundant taxa [@Shade2014], and this concept would likely apply to bacterial groups recovering population density following cessation of antibiotic pretreatment. As such, *C. difficile* may preferentially seek to compete with these organisms to ultimately affect greater change to the entire ecosystem and open a long-lasting nutrient niche. While this hypothesis requires further exploration, it provides an ecological framework to study the interactions between *C. difficile* and members of susceptible communities.

This study is one of the first *in vivo* observations that a medically relevant bacterial pathogen may alter the metabolic activity of a host-associated community to promote its own colonization. This is also the first application of metatranscriptomic analysis of the gut microbiota *in vivo* and in response to a pathogen. Other groups have identified potential metabolite markers of *C. difficile* infection in patient feces [@Rojo2015], but they were not able to make correlations with changes in community metabolism that were afforded to us by paired untargeted metabolomic analysis and metatranscriptomic sequencing. In a recent study, a tick-vectored bacterial pathogen altered the ability of the resident microbiota of the tick by interrupting proper biofilm formation and allowing lasting colonization [@Abraham2017]. Furthermore, it was recently found that bacterial metabolic generalists may be more likely to actively antagonize the growth of other species in an environment which they are colonizing [@Russel2017]. In previous work from our group, we found that *C. difficile* has a wide nutrient niche-space *in vivo* and is most likely utilizes its role as a metabolic generalist to colonize diverse gut microbiomes [@Jenior2017]. The ability to simultaneously antagonize the metabolism of surrounding species in cecal environments that support persistence would explain the more significant shifts in the metatranscriptome. While we acknowledge that this study may not elucidate the specific mechanism by which this interaction occurs, the combined systems analysis strengthens each individual level of observation and only when employed together does a clearer definition of *C. difficile*-related microbial ecology in the gut emerge. This research lays the groundwork for a more rationale consideration of the metabolic functionalities of bacterial taxa to consider when attempting to rebuild *C. difficile* colonization resistance across differentially perturbed gut environments.

In spite of consistent signals across multiple levels of -omics datasets, possible shortcomings to our interpretation of the presented data do exist. First, as with all transcriptomic studies, the relative level of mRNA detected for a given gene does not necessarily reflect the amount of functional protein made by a cell. Furthermore, interpretation of timing may also be an issue since a large influx of transcript for a specific product may signal an initial transcription before subsequent translation has been able to occur. This also omits consideration of any post-translational modifications that are required for ultimately functional enzymes. Additionally, due to the low relative abundance of *C. difficile* in these communities as well as the high density of other surrounding bacterial species required focusing sequencing efforts on depth rather than multiple replicates within each group. Greater transcript read abundance per gene allows for improved survey for the activity of lowly abundant species as well as greater confidence in genes found to be highly transcribed. Although the lack of animal-based replication for the metatranscriptomic data does potentially limit the ability to generalize our results, this approach has been successfully utilized by numerous groups in the past to accurately characterize transcriptionally activity across communities of bacteria [@Sheik2014;@Franzosa2014;@Jorth2014;@Jenior2017]. In terms of metabolomics, alternative possible interpretations of the data also exist. For example, we considered metabolites that did not change in concentration between uninfected and infected conditions as not impacted by changes in bacterial metabolism induced by *C. difficile* colonization. However, this may instead indicate that the metabolism of *C. difficile* itself may simply replace the level present in the uninfected community. Such instances may not be detectable through untargeted mass spectrometry alone, however the combination of methods utilized here present a much more unified description of the system than any of the component techniques alone.

Several groups have attempted to identify single bacterial species or limited strain consortia that are able to replicate this effect, but each has been met with incomplete restoration of colonization resistance or function [@Reeves2012;@Lawley2012;@Petrof2013;@Buffie2014]. The effect of *C. difficile* colonization on community metabolic activity could also be linked to pathogen strain and may offer an explanation to the propensity of some strains to persist over others [@Adlerberth2014]. Moreover, the current work contributes to the existing concept that the healthy gut microbiota maintains colonization resistance to *C. difficile* by outcompeting the pathogen for preferred nutrient niche space. Ultimately, our results suggest that each susceptible and subsequently infected microbiome may be unique and require specific microbes or functionalities to restore colonization resistance against *C. difficile* in that specific context. Conversely, colonization resistance against *C. difficile* may be the result of contributions by distinct subcommunities of bacteria across each unique resistant gut community. As the microbiome is so intimately connected to colonization resistance against the bacterium, it has become imperative to understand what factors allow some gut environments to be persistently colonized while others are not. This research lays the groundwork for future studies to assess context dependent restoration of *C. difficile* colonization resistance and what factors are able to interfere with the ability of *C. difficile* to modify gut ecology in order to promote clearance.


**Materials and Methods**

**Animal care and antibiotic administration.** For a more detailed description of the animal model, refer to Jenior et al. [@Jenior2017]. Briefly, approximately equal numbers of male and female conventionally-reared six-to-eight week-old C57BL/6 mice were randomly assigned to each experimental group (genders were housed separately). Nine mice were used in each experimental and control group. They were administered one of three antibiotics; cefoperazone, streptomycin, or clindamycin before oral *C. difficile* infection (Table S1). All animal protocols were approved by the University Committee on Use and Care of Animals at the University of Michigan and carried out in accordance with the approved guidelines from the Office of Laboratory Animal Welfare (OLAW), United States Department of Agriculture (USDA) registration, and the Association for Assessment and Accreditation of Laboratory Animal Care (AAALAC). The protocol license Institutional Animal Care and Use Committee (IACUC) number for all described experiments is PRO00006983.

***C. difficile* infection and necropsy.** On the day of challenge, 1x10^3^ *C. difficile* spores were administered to mice via oral gavage in phosphate-buffered saline (PBS) vehicle. Mock-infected animals were given an oral gavage of 100 ul PBS at the same time as those mice administered *C. difficile* spores. 18 hours following infection, mice were euthanized by CO~2~ asphyxiation and necropsied to obtain the cecal contents. Aliquots were immediately flash frozen for later DNA extraction and toxin titer analysis. A third aliquot was transferred to an anaerobic chamber for quantification of *C. difficile* abundance. The remaining content in the ceca was mixed in a stainless steel mortar housed in a dry ice and ethanol bath. Cecal contents from all mice within each pretreatment group were pooled into the mortar prior to grinding to a fine powder. The ground content was then stored at -80&deg;C for subsequent RNA extraction. For 10-day colonization studies, fresh stool was collected from infected mice each day beginning on the day of infection. Mice were monitored for overt signs of disease and were euthanized after the final stool collection.

***C. difficile* cultivation and quantification.** Cecal samples were weighed and serially diluted under anaerobic conditions with anaerobic PBS. Differential plating was performed to quantify both *C. difficile* spores and vegetative cells by plating diluted samples on CCFAE plates (fructose agar plus cycloserine, cefoxitin, and erythromycin) at 37&deg;C for 24 hours under anaerobic conditions [@Wilson1982]. In parallel, undiluted samples were heated at 60&deg;C for 30 minutes to eliminate vegetative cells and enrich for spores [@Sorg2010]. These samples were serially diluted under anaerobic conditions in anaerobic PBS and plated on CCFAE with taurocholate at 37&deg;C for 24 hours. Plating was simultaneously done for heated samples on CCFAE to ensure all vegetative cells had been eliminated. Quantification of *C. difficile* cfu for the 10-day colonization experiments was performed from stool using TCCFAE to measure total *C. difficile* load in these animals over time.

***C. difficile* toxin titer assay.** To quantify the titer of toxin in the cecum a Vero cell (ATCC CCL-81) rounding assay was performed as described elsewhere [@Leslie2015;@Jenior2017]. Briefly, filtered-sterilized cecal content was serially diluted in PBS and added to Vero cells in a 96-well plate where it was then incubated at 37&deg;C for 24 hours. Plates were then inspected for evidence of cell rounding.

**DNA/RNA extraction and sequencing library preparation.** DNA for shotgun metagenomic and 16S rRNA gene sequencing was extracted from approximately 50 mg of cecal content from each mouse using the PowerSoil-htp 96 Well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf). The V4 region of the bacterial 16S rRNA gene was amplified using custom barcoded primers [@Kozich2013]. Equal molar ratios of raw isolated DNA within each treatment group were then pooled and ~2.5 ng of material was used to generate shotgun libraries with a modified 10-cycle Nextera XT genomic library construction protocol (Illumina). This was done to mimic the pooling strategy necessary for metatranscriptomic library preparation. Final libraries were pooled at equal molar ratios and stored at -20&deg;C. For RNA extraction, a more detailed description of the procedure can be found in [@Jenior2017]. Briefly, immediately before RNA extraction, 3 ml of lysis buffer (2% SDS, 16 mM EDTA and 200 mM NaCl) contained in a 50 ml polypropylene conical tube was heated for 5 minutes in a boiling water bath [@Lopez-Medina2011]. The hot lysis buffer was added to the frozen and ground cecal content. The mixture was boiled with periodic vortexing for another 5 minutes. After boiling, an equal volume of 37&deg;C acid phenol/chloroform was added to the cecal content lysate and incubated at 37&deg;C for 10 minutes with periodic vortexing. The mixture was the centrifuged at 2,500 x g at 4&deg;C for 15 minutes. The aqueous phase was then transferred to a sterile tube and an equal volume of acid phenol/chloroform was added. This mixture was vortexed and centrifuged at 2,500 x g at 4&deg;C for 5 minutes. The process was repeated until aqueous phase was clear. The last extraction was performed with chloroform/isoamyl alcohol to remove acid phenol. An equal volume of isopropanol was added and the extracted nucleic acid was incubated overnight at -20&deg;C. The following day the sample was centrifuged at 12000 x g at 4&deg;C for 45 minutes. The pellet was washed with 0&deg;C 100% ethanol and resuspended in 200 ul of RNase-free water. Following the manufacturer's protocol, samples were then treated with 2 ul of Turbo DNase for 30 minutes at 37&deg;C. RNA samples were retrieved using the Zymo Quick-RNA MiniPrep according the manufacturer's protocol. The Ribo-Zero Gold, immediately before RNA extraction, 3 ml of lysis buffer (2% SDS, 16 mM EDTA and 200 mM NaCl) contained in a 50 ml polypropylene conical tube was heated for 5 minutes. RNA Removal Kit Epidemiology was then used to deplete prokaryotic and eukaryotic rRNA from the samples according the manufacturer's protocol. Stranded RNA-Seq libraries were made constructed with the TruSeq Total RNA Library Preparation Kit v2, both using the manufacturer's protocol. Completed libraries were pooled in equal molar ratios within their respective groups and stored at -20&deg;C until time of sequencing.

**High-throughput sequencing and raw read curation.** Sequencing of 16S rRNA gene amplicon libraries was performed using an Illumina MiSeq sequencer as described previously [@Kozich2013]. The 16S rRNA gene sequences were curated using the mothur software package (v1.36) as described in [@Jenior2017]. Shotgun metagenomic sequencing was performed in 2 phases. Libraries from mock-infected communities, that were also to be utilized for *de novo* contig assembly, were sequenced using an Illumina HiSeq 2500 on 2x250 paired-end settings and was repeated across 2 lanes to normalize for inter-run variation. *C. difficile*-infected metagenomic libraries were sequenced with an Illumina NextSeq 300 with 2x150 settings across 2 runs to also normalize for inter-run variation. These efforts resulted in an average of 280,000,000 paired raw reads per sample. Metatranscriptomic sequencing was performed on an Illumina HiSeq 2500 with 2x50 settings and was repeated across 4 lanes for normalization and to normalize for technical variation between lans and to obtain necessary coverage [@Franzosa2014]. This gave an average of 380,000,000 raw cDNA reads per library. Both metagenomic and metatranscriptomic sequencing was performed at the University of Michigan Sequencing Core. Raw sequence read curation for both metagenomic and metatranscriptomic datasets was performed in a two step process. Residual 5-prime and 3-prime Illumina adapter sequences were trimmed using CutAdapt [@Martin2011] on a per library basis. Reads were quality trimmed using Sickle [@Joshi2011] with a quality cutoff of Q30. This resulted in approximately 270 million reads per library (both paired and orphaned) for both metagenomic and metatranscriptomic sequencing. Actual read abundances for individual metagenomic and metatranscriptomic sequencing efforts can be found in Table S2.

**Metagenomic contig assembly and gene annotation.** Metagenomic contigs were assembled using Megahit [@Li2014] with the following settings; minimum kmer size of 87, maximum kmer size of 127, and a kmer step size of 10. Prodigal was utilized to to identify putative gene sequences, and were screened for a minimum length of 250 nucleotides. These sequences were translated to amino acids and peptides were annotated against the KEGG protein database [@Ogata1999] using Diamond implementation of BLASTp [@Buchfink2015]. Peptide-level gene annotations were assigned to the corresponding nucleotide sequence, and genes failing to find a match in KEGG were preserved as unannotated genes. Final nucleotide fasta files with KEGG annotations were then utilized in the construction of Bowtie2 [@Langmead2012] mapping databases from downstream analyses.

**DNA/cDNA read mapping and normalization.** Mapping was accomplished using Bowtie2 [@Langmead2012] and the default stringent settings. Optical and PCR duplicates were then removed using Picard MarkDuplicates (http://broadinstitute.github.io/picard/). The remaining mappings were converted to idxstats format using Samtools [@Li2009] and the read counts per gene were tabulated. Discordant pair mappings were discarded and counts were then normalized to read length and gene length to give a per base report of gene coverage. Transcript abundance was then normalized to gene abundance to yield overall level of transcription for each gene. Unless indicated otherwise, each collection of reads was then 1000-fold iteratively subsampled to 90% of the lowest sequence total within each analysis, and a median transcription value for each gene was calculated. Reads contributed by *C. difficile* were removed from analysis using Bowtie2 against the *C. difficile* str. 630 genome with settings allowing for up to 2 mismatches.

**Quantification of *in vivo* metabolite relative concentrations.** Metabolomic analysis was performed by Metabolon (Durham, NC), for a detailed description of the procedure refer to [@Jenior2017]. Briefly, all methods utilized a Waters ACQUITY ultra-performance liquid chromatography (UPLC) and a Thermo Scientific Q-Exactive high resolution/accurate mass spectrometer interfaced with a heated electrospray ionization (HESI-II) source and Orbitrap mass analyzer at 35,000 mass resolution. Samples were dried then reconstituted in solvents compatible to each of the four methods. The first, in acidic positive conditions using a C18 column (Waters UPLC BEH C18-2.1x100 mm, 1.7 um) using water and methanol, containing 0.05% perfluoropentanoic acid (PFPA) and 0.1% formic acid (FA). The second method was identical to the first but was chromatographically optimized for more hydrophobic compounds. The third approach utilized a basic negative ion optimized conditions using a separate dedicated C18 column. Basic extracts were gradient eluted from the column using methanol and water, however with 6.5mM Ammonium Bicarbonate at pH 8. Samples were then analyzed via negative ionization following elution from a hydrophilic interaction chromatography column (Waters UPLC BEH Amide 2.1x150 mm, 1.7 um) using a gradient consisting of water and acetonitrile with 10mM Ammonium Formate, pH 10.8. The MS analysis alternated between MS and data-dependent MS n scans using dynamic exclusion. The scan range varied slighted between methods but covered 70-1000 m/z. Library matches for each compound were checked for each sample and corrected if necessary.

**Statistical methods.** All statistical analyses were performed using R (v.3.2.0). Significant differences between community structure of treatment groups from 16S rRNA gene sequencing were determined with AMOVA in the mothur software package. Significant differences of inverse Simpson diversity, cfu, toxin titer, and metabolite concentrations were determined by Wilcoxon signed-rank test with Benjamini-Hochberg correction using a study-wide Type I error rate of 0.05. Undetectable points used half the limit of detection for cfu and toxin statistical calculations. LEfSe analysis with OTU data was performed with a cutoff that each OTU must appear in all samples from their respective groups to be considered true signal (n=9). Random forest was performed using the AUCRF implementation [@Calle2011] as well as the standard package [@Breiman2001] in R . Distances of outlier points from center line during metatranscriptomic comparisons was accomplished using 2-dimensional linear geometry.


**Acknowledgments**

The authors would like to acknowledge Charles Koumpouras for assistance with DNA extractions and metabolomic sample preparation. We would also like to acknowledge members of the University of Michigan Germfree Mouse Center, University of Michigan Sequencing Core, and Metabolon for their assistance in experimental design, execution, and data collection. Thank you also to Greg Medlock and Maureen Carey for input on details of the analysis.

**Author Affiliations**
**Department of Microbiology and Immunology, University of Michigan, Ann Arbor, Michigan.**
Matthew L. Jenior Ph.D., Jhansi L. Leslie Ph.D., & Patrick D. Schloss Ph.D.

**Department of Internal Medicine/Infectious Diseases Division, University of Michigan Medical Center, Ann Arbor, Michigan.**
**Department of Microbiology and Immunology, University of Michigan, Ann Arbor, Michigan.**
Vincent B. Young M.D. Ph.D.

**Author Contributions**
M.L.J. conceived, designed and performed experiments, analyzed data, and drafted the manuscript. J.L.L. performed experiments, analyzed data, and contributed to the manuscript. V.B.Y. contributed to the manuscript. P.D.S. interpreted data and contributed the manuscript.
The authors declare no conflicts of interest.

**Corresponding author**
Correspondence to [Patrick D. Schloss](pschloss@umich.edu)


**Data Availability**

Pooled and quality trimmed transcriptomic (SRA; PRJNA354635) and 16S rRNA gene amplicon read data (SRA; PRJNA383577) from infection experiments are available through the NCBI Sequence Read Archive. Metagenomeic reads, mock-infected metatranscriptomic reads can be found also on the SRA (PRJNA415307). Data processing steps for beginning from raw sequence data to the final manuscript are hosted at https://github.com/mjenior/Jenior_Metatranscriptomics_mSphere_2018.


**Figure and Table Legends**

**Figure 1 | Experimental models of *C. difficile* infection and distinct virulence patterns.** **(A)** Experimental time lines of antibiotic pretreatment and infection mouse models. **(B)** *C. difficile* 630 cfu in stool of infected mice following each antibiotic pretreatment regimen over 10 days of infection. Median and interquartile range are shown for each time point. **(C)** Quantification of *C. difficile* cfu and toxin titer in cecal content across antibiotic treatment models after 18 hours of infection. Black lines indicate median values. Gray asterisks indicate significant difference from no antibiotic controls by Wilcoxon rank-sum test with the Benjamini-Hochberg correction (all *p* < 0.001). Points below the limit of detection were excluded from statistical testing. Nine mice were used in each experimental and control group and samples from these animals were used for all downstream analyses.

**Figure 2 | Impact of *C. difficile* colonization on gut microbiota community composition.** **(A)** Relative abundance of family-level taxonomic classification for OTUs in each pretreatment group. **(B-D)** Discriminating OTUs with LEfSe analysis between Mock and *C. difficile*-infected communities within each pretreatment group, *p*-values from LEfSe are shown on the left with taxonomic information (all *p* < 0.05). Relative abundance of the respective OTUs from each mouse along with medians are shown for each pretreatment group. 

**Figure 3 | Metabolomic markers of infection susceptibility.** Results from UPLC/MS metabolomic analysis of cecal content from mice also used in previous analyses. Only mock-infected metabolomic results were included this analysis to identify markers of susceptibility. **(A-B)** NMDS ordinations of Bray-Curtis distances comparing the cecal metabolome dissimilarity of mice receiving no pretreatment or one of the three distinct classes of antibiotics. (A) Metabolomes of resistant mice are significantly different from antibiotic pretreated animals (*p* < 0.001). (B) Antibiotic pretreated metabolomes and also significantly distinct from one another (*p* < 0.001). Significant differences for NMDS analyses were calculated with AMOVA. **(C-D)** Scaled intensities for minimum subset of metabolites from random forest feature selection that correctly discriminate all groups from A-B (Out-of-bag error = 0%). (C) Metabolites that distinguish cecal content from mice resistant to *C. difficile* colonization from susceptible (antibiotis-pretreated) animals. (D) Metabolites that differentiate antibiotic pretreatment groups. Asterisks indicate significant difference by Wilcoxon rank-sum test with Benjamini-Hochberg correction.

**Figure 4 | Differential effects of CDI on the cecal metabolome.** **(A-C)** NMDS ordinations of Bray-Curtis distances comparing infection groups within each antibiotic pretreatment; (A) Streptomycin, (B) Cefoperazone, and (C) Clindamycin. Significant differences were determined by AMOVA. **(D-F)** Relative concentrations of metabolites with highest Mean Decrease Accuracy (MDA) from Random Forest feature selection to discriminate between Mock and *C. difficile*-infected conditions within each antibiotic pretreatment. Respective MDA is labeled in brackets next to each metabolite name and out of bag error from internal cross-validation of each model is labeled under the plotting area. (D) Streptomycin pretreatment, (E) Cefoperazone pretreatment, and (F) Clindamycin pretreatment. **(G)** Relative concentrations of 5-aminovalerate across all experimental and control groups. Significant differences are labeled along the right axis and were calculated with Wilcoxon rank-sum test with Benjamini-Hochberg correction.

**Figure 5 | Genes and pathways with largest degree of change in metatranscriptome associated with CDI.** Venn diagrams showing the overlap in top 15% of differentially transcribed individual genes **(A)** or pathways **(B)** respectively in each antibiotic preatretment between CDI and mock infections. Below each venn diagram are the corresponding genes/pathways contained in the central shared regions for all three pretreatments. Solid colored bars indicated *C. difficile* infection and dashed bars are mock infected, separate colors denoted sntibiotic pretreatment. Genes/pathways are ranked in terms of decreasing sample variance between all analyzed groups. Genes are also labeled with numbers indicated membership in KEGG pathways (pathway legend below).

**Figure 6 | *C. difficile* colonization alters gene transcription of taxonomic groups differentially between antibiotic pretreatments.** **(A-C)** Each point represents a unique gene from the respective metagenomic assembly. Coordinates were determined by the log~2~-transformed transcription level of each gene between *C. difficile*-infected and mock-infected conditions. Metatranscriptomic read abundances were normalized to their associated metagenomic coverage. Outliers were defined using linear correlation and a squared residual cutoff of 2. Distance of outliers to the x=y line were also calculated and represented in unites of deviation or UD. Colored indicate genus of origin, and gray areas denote genes with consistent transcription between conditions (defined by outlier analysis). Antibiotic pretreatments; (A) Streptomycin-pretreated, (B) Cefoperazone-pretreated, and (C) Clindamycin-pretreated. **(D)** Absolute difference in metatranscriptomic reads contributed by each genus in pretreatments between mock and *C. difficile*-infected conditions. Colored lines denoted antibiotic pretreatment. Each point represents all transcript contributed by that genus in each pretreatment group. Numbers at the base of pretreatment lines in the 0.1% panel represent the quantity of genera in each group as some points are obscured.

**Supplementary Figure 1 | Impact of antibiotic pretreatment and *C. difficile* infection on cecal bacterial community structure.** **(A)** Inverse-Simpson diversity of cecal communities from all pretreatment groups. Gray stars indicate significant difference from no antibiotic controls (all *p* < 0.001). Black stars denote within pretreatment group significant difference between mock and *C. difficile*-infected communities. Differences were calculated using Wilcoxon rank-sum test with Benjamini-Hochberg correction. NMDS ordinations of Bray-Curtis distances comparing the groups labeled in the bottom left of each plotting area. **(B)** Antibiotic-pretreated compared to Untreated controls and **(C)** comparison between only antibiotic pretreatment groups. **(D-F)** Within antibiotic pretreatment comparisons for the effect of *C. difficile* colonization on community structure. Significant differences and correlation coefficients for ordination analyses were found using ANOSIM.

**Supplementary Figure 2 | Relative concentrations of select metabolite groups in each group** Metabolites included in this analysis were chosen based on their previously published links to *C. difficile* physiology or susceptibility to infection. Groups are as follows; **(A)** Bile acids, **(B)** Carbohydrates, and **(C)** Amino Acids. Significant differences were determined by Wilcoxon rank-sum test with Benjamini-Hochberg correction.

**Supplementary Figure 3 | Effect of infection on the cecal metabolome across pretreatment groups** Pooled analysis of antibiotic pretreated animals only. **(A)** NMDS ordination of Bray-Curtis distances differentiating mock and *C. difficile*-infected metabolomes (*p* = 0.075). **(B)** Random forest classification results for metabolites that effectively distinguish infected and uninfected conditions. Shown are relative concentrations of the top 10 metabolites with the highest mean decrease in accuracy from internal cross-validation. Shown in the top right corner of each panel are the metabolite names and mean decrease accuracy of each. Significant differences in concentration between mock and *C. difficile*-infected groups were determined by Wilcoxon rank-sum test with Benjamini-Hochberg correction.

**Supplementary Figure 4 | Largest discrete gene and pathway, metatranscriptomic differences between antibiotic pretreatments and resistant microbiota**  Included are the top 10 genes from unique subsections of venn diagrams from each pretreatment group in Figure 5A-B. Genes/Pathways with the largest disparity compared the metatranscriptome from the cecal community of untreated mice. Panels for gene-level analysis are as follows; **(A)** Streptomycin-pretreatment, **(B)** Cefoperazone-pretreatment, and **(C)** Clindamycin-pretreatment. **(D)** Pathway-level annotations for unique groupings in Figure 5B.

**Supplementary Figure 5 | Unique genes with functional annotation detectable within each metagenome and metatranscriptome** Genes in each datasets were derived from respective metagenomic assemblies, with only those genes that mapped to a KEGG pathway-level annotation. Results are shown as follows: **(A)** Untreated, **(B)** Streptomycin-pretreated, **(C)** Cefoperazone-pretreated, and **(D)** Clindamycin-pretreated mice. Each panel includes that treatments' unique genes from metagenomic assembly and genes that recruited at least one cDNA read from the corresponding metatranscriptomes. Collector's curves from rarefaction analysis of reads mapped to genes from **(E)** metagenomes and **(F)** metatranscriptomes.

**Supplementary Table 1 | Chosen antibiotic pretreatment regimes** Antibiotic classes, mechanisms, and dosage information for each pretreatment.

**Supplementary Table 2 | High-throughput sequencing read counts and metagenomic assembly quality** Raw and curated read abundances for both metagenomic and metatranscriptomic sequencing efforts. Raw read curation steps are outlined in Materials & Methods. Metagenomic contig summary statistics reflect the quality of assembly for each group.

**Supplementary Table 3 | Normalized cDNA abundances of complete metatranscriptomes for each antibiotic pretreated group**
Final cDNA read abundances for each gene from the corresponding metagenomic assembly. Normalization reflects standardizing cDNA read abundances to both read length and target gene length as well as metagenomic coverage for each gene.

**Supplementary Table 4 | Median scaled intensity of metabolites for each antibiotic pretreatment and infection group**

**Supplementary Table 5 | Summary statistics for datasets with replication generated during this study**

**References**
